-- MySQL dump 10.13  Distrib 5.1.66-MariaDB, for pc-linux-gnu (i686)
--
-- Host: localhost    Database: TERMINOLOGY_DB
-- ------------------------------------------------------
-- Server version	5.1.66-MariaDB-mariadb118

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `CITATION`
--

DROP TABLE IF EXISTS `CITATION`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `CITATION` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `DOI` varchar(255) DEFAULT NULL,
  `YEAR` int(11) DEFAULT NULL,
  `SOURCE` text,
  `STR` text NOT NULL,
  PRIMARY KEY (`ID`),
  UNIQUE KEY `UQ_CITATION_ID` (`ID`)
) ENGINE=MyISAM AUTO_INCREMENT=128 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `CITATION`
--

LOCK TABLES `CITATION` WRITE;
/*!40000 ALTER TABLE `CITATION` DISABLE KEYS */;
INSERT INTO `CITATION` VALUES (1,NULL,NULL,NULL,'Jorgensen WL, Tirado-Rives J (1988). \'The OPLS Force Field for Proteins. Energy Minimizations for Crystals of Cyclic Peptides and Crambin\'. J. Am. Chem. Soc. 110 (6): 1657-1666.'),(2,NULL,NULL,NULL,'Mahoney & Jorgensen (2000). J. Chem. Phys. 112, 8910-8922'),(3,NULL,NULL,NULL,'Jorgensen, W. L.; Madura, J. D.; Swenson, C. J. J. Am. Chem. Soc. 1984, 106, 6638.'),(4,NULL,NULL,NULL,'H.J.C. Berendsen, J.P.M. Postma, W.F. van Gunsteren, and J. Hermans, In Intermolecular Forces, edited by B. Pullman (Reidel, Dordrecht, 1981), p. 331.'),(5,NULL,NULL,NULL,'H. J. C. Berendsen, J. R. Grigera, and T. P. Straatsma (1987). The missing term in effective pair potentials. Journal of Physical Chemistry 91: 6269-6271.'),(6,NULL,NULL,NULL,'J. L. F. Abascal, E. Sanz, R. GarcÃ­a FernÃ¡ndez, and C. Vega. A potential model for the study of ices and amorphous water: TIP4P/Ice. J. Chem. Phys. 2005, 122, 234511.'),(7,NULL,NULL,NULL,'J. L. F. Abascal and C. Vega. A general purpose model for the condensed phases of water: TIP4P/2005. J. Chem. Phys. 2005, 123, 234505.'),(8,NULL,NULL,NULL,'H. W. Horn, W. C. Swope, J. W. Pitera, J. D. Madura, T. J. Dick, G. L. Hura, and T. Head-Gordon. Development of an improved four-site water model for biomolecular simulations: TIP4P-Ew. J. Chem. Phys. 2004, 120, 9665-9678.'),(9,NULL,NULL,NULL,'H. J. C. Berendsen, J. R. Grigera, and T. P. Straatsma. The Missing Term in Effective Pair Potentials. J. Phys. Chem 1987, 91, 6269-6271.'),(10,NULL,NULL,NULL,'AllnÃ©r, Olof, Lennart Nilsson, and Alessandra Villa. \'Magnesium ion-water coordination and exchange in biomolecular simulations.\' Journal of Chemical Theory and Computation 8.4 (2012): 1493-1502.'),(11,NULL,NULL,NULL,'Joung, In Suk, and Thomas E. Cheatham III. \'Determination of alkali and halide monovalent ion parameters for use in explicitly solvated biomolecular simulations.\' The Journal of Physical Chemistry B 112.30 (2008): 9020-9041.'),(12,NULL,NULL,NULL,'Beglov, Dmitrii, and Benoit Roux. \'Finite representation of an infinite bulk system: solvent boundary potential for computer simulations.\' The Journal of chemical physics 100 (1994): 9050.'),(13,NULL,NULL,NULL,'Aqvist, Johan. \'Ion-water interaction potentials derived from free energy perturbation simulations.\' Journal of Physical Chemistry 94.21 (1990): 8021-8024.'),(14,NULL,NULL,NULL,'Li, Pengfei, et al. \'Rational Design of Particle Mesh Ewald Compatible Lennard-Jones Parameters for+ 2 Metal Cations in Explicit Solvent.\' Journal of Chemical Theory and Computation (2013).'),(15,NULL,NULL,NULL,'Brooks, Bernard R., et al. \'CHARMM: the biomolecular simulation program.\' Journal of computational chemistry 30.10 (2009): 1545-1614.'),(16,NULL,NULL,NULL,'Case, David A., et al. \'The Amber biomolecular simulation programs.\' Journal of computational chemistry 26.16 (2005): 1668-1688.'),(17,NULL,NULL,NULL,'Hamelberg D, de Oliveira CA, McCammon JA, \'Sampling of slow diffusive conformational transitions with accelerated molecular dynamics\'. J Chem Phys. 2007 Oct 21; 127(15):155102'),(18,NULL,NULL,NULL,'HÃ©nin, J., Chipot, C., \'Overcoming free energy barriers using unconstrained molecular dynamics simulations.\' The Journal of chemical physics 121 (2004): 2904.'),(19,NULL,NULL,NULL,'http://www.cp2k.org/'),(20,NULL,NULL,NULL,'VandeVondele J, Rothlisberger U. Canonical adiabatic free energy sampling (CAFES): a novel method for the exploration of free energy surface. J Phys Chem B. 2002;106:203-8.'),(21,NULL,NULL,NULL,'J. M. Tao, J. P. Perdew, V. N. Staroverov, and G. E. Scuseria, \'Climbing the density functional ladder: Nonempirical meta-generalized gradient approximation designed for molecules and solids,\' Phys. Rev. Lett., 91 (2003) 146401.'),(22,NULL,NULL,NULL,'Borrero EE, Escobedo FA, \'Optimizing the sampling and staging for simulations of rare events via forward flux sampling schemes\'. J Chem Phys. 2008 Jul 14; 129(2):024115.'),(23,NULL,NULL,NULL,'T. Van Voorhis and G. E. Scuseria, \'A never form for the exchange-correlation energy functional,\' J. Chem. Phys., 109 (1998) 400-10.'),(24,NULL,NULL,NULL,'Roitberg A, Elber R. Modeling of side chains in peptides and proteins: application of the locally enhanced sampling and the simulated annealing methods to find minimum energy conformations. J Chem Phys. 1991;95:9277-87.'),(25,NULL,NULL,NULL,'A. D. Boese and N. C. Handy, \'New exchange-correlation density functionals: The role of the kinetic-energy density,\' J. Chem. Phys., 116 (2002) 9559-69.'),(26,NULL,NULL,NULL,'Transition networks for modeling the kinetics of conformational change in macromolecules. NoÃ© F, Fischer S, Curr Opin Struct Biol. 2008 Apr; 18(2):154-62.'),(27,NULL,NULL,NULL,'Huo S, Straub JE. The MaxFlux algorithm for calculating variationally optimized reaction paths for conformational transitions in many body systems at finite temperature. J Chem Phys. 1997;107:5000ï¾–6. doi: 10.1063/1.474863.'),(28,NULL,NULL,NULL,'Parrinello M. Eppur si muove. In: Zewail AH, editor. Physical Biology: 4D Visualization of Complexity. London: Imperial College Press; 2008. pp. 247-66.'),(29,NULL,NULL,NULL,'Elber R, \'A milestoning study of the kinetics of an allosteric transition: atomically detailed simulations of deoxy Scapharca hemoglobin\'. Biophys J. 2007 May 1; 92(9):L85-7.'),(30,NULL,NULL,NULL,'X. Xu and W. A. Goddard III, \'The X3LYP extended density functional for accurate descriptions of nonbond interactions, spin states, and thermochemical properties,\' Proc. Natl. Acad. Sci. USA, 101 (2004) 2673-77.'),(31,NULL,NULL,NULL,'Henkelman G, JÃ³nsson H. Improved tangent estimate in the nudged elastic band method for finding minimum energy paths and saddle points. J Chem Phys. 2000;113:9978-85.'),(32,NULL,NULL,NULL,'Isralewitz, Barry, Mu Gao, and Klaus Schulten. \'Steered molecular dynamics and mechanical functions of proteins.\' Current opinion in structural biology 11.2 (2001): 224-230.'),(33,NULL,NULL,NULL,'Solvent coarse-graining and the string method applied to the hydrophobic collapse of a hydrated chain. Miller TF 3rd, Vanden-Eijnden E, Chandler D, Proc Natl Acad Sci U S A. 2007 Sep 11; 104(37):14559-64.'),(34,NULL,NULL,NULL,'Minary P, Tuckerman ME, Martyna GJ. Dynamical spatial warping: a novel method for the conformational sampling of biophysical structure. SIAM J Sci Comput. 2008;30:2055-83'),(35,NULL,NULL,NULL,'Sugita Y, Okamoto Y. Replica-exchange molecular dynamics methods for protein folding. Chem Phys Lett. 1999;314:141-51. doi: 10.1016/S0009-2614(99)01123-9'),(36,NULL,NULL,NULL,'Dellago C, Bolhuis PG. Transition path sampling simulations of biological systems. Top Curr Chem. 2007;268:291-317.'),(37,NULL,NULL,NULL,'Kong Y, Shen Y, Warth TE, Ma J, \'Conformational pathways in the gating of Escherichia coli mechanosensitive channel\', Proc Natl Acad Sci U S A. 2002 Apr 30; 99(9):5999-6004.'),(38,NULL,NULL,NULL,'Y. Zhao and D. G. Truhlar, \'Density Functional for Spectroscopy: No Long-Range Self-Interaction Error, Good Performance for Rydberg and Charge-Transfer States, and Better Performance on Average than B3LYP for Ground States,\' J. Phys. Chem. A, 110 (2006) 13126-30.'),(39,NULL,NULL,NULL,'http://classic.chem.msu.su/gran/gamess/'),(40,NULL,NULL,NULL,'Valiev, M.; Bylaska, E.J.; Govind, N.; Kowalski, K.; Straatsma, T.P.; Van Dam, H.J.J.; Wang, D.; Nieplocha, J. et al. (2010). \'NWChem: A comprehensive and scalable open-source solution for large scale molecular simulations\'. Computer Physics Communications 181 (9): 1477-1489.'),(41,NULL,NULL,NULL,'James C. Phillips, Rosemary Braun, Wei Wang, James Gumbart, Emad Tajkhorshid, Elizabeth Villa, Christophe Chipot, Robert D. Skeel, Laxmikant Kale, and Klaus Schulten, \'Scalable molecular dynamics with NAMD\', Journal of Computational Chemistry, 26:1781-1802, 2005.'),(42,NULL,NULL,NULL,'http://openmopac.net'),(43,NULL,NULL,NULL,'Y. Zhao and D. G. Truhlar, \'The M06 suite of density functionals for main group thermochemistry, thermochemical kinetics, noncovalent interactions, excited states, and transition elements: two new functionals and systematic testing of four M06-class functionals and 12 other functionals,\' Theor. Chem. Acc., 120 (2008) 215-41.'),(44,NULL,NULL,NULL,'Y. Zhao, N. E. Schultz, and D. G. Truhlar, \'Design of density functionals by combining the method of constraint satisfaction with parametrization for thermochemistry, thermochemical kinetics, and noncovalent interactions,\' J. Chem. Theory and Comput., 2 (2006) 364-82.'),(45,NULL,NULL,NULL,'Y. Zhao, N. E. Schultz, and D. G. Truhlar, \'Exchange-correlation functional with broad accuracy for metallic and nonmetallic compounds, kinetics, and noncovalent interactions,\' J. Chem. Phys., 123 (2005) 194101: 1-18.'),(46,NULL,NULL,NULL,'http://www.gaussian.com/'),(47,NULL,NULL,NULL,'Hess, Berk, et al. \'GROMACS 4: Algorithms for highly efficient, load-balanced, and scalable molecular simulation.\' Journal of chemical theory and computation 4.3 (2008): 435-447.'),(48,NULL,NULL,NULL,'http://www.schrodinger.com/productpage/14/7/'),(49,NULL,NULL,NULL,'Plimpton, S., P. Crozier, and A. Thompson. \'LAMMPS-large-scale atomic/molecular massively parallel simulator.\' Sandia National Laboratories (2007).'),(50,NULL,NULL,NULL,'Christen, Markus, et al. \'The GROMOS software for biomolecular simulation: GROMOS05.\' Journal of computational chemistry 26.16 (2005): 1719-1751.'),(51,NULL,NULL,NULL,'T. M. Henderson, A. F. Izmaylov, G. Scalmani, and G. E. Scuseria, \'Can short-range hybrids describe long-range-dependent properties?,\' J. Chem. Phys., 131 (2009) 044108.'),(52,NULL,NULL,NULL,'A. D. Boese and N. C. Handy, \'A new parametrization of exchange-correlation generalized gradient approximation functionals,\' J. Chem. Phys., 114 (2001) 5497-503.'),(53,NULL,NULL,NULL,'http://www.biomolecular-modeling.com/Abalone/'),(54,NULL,NULL,NULL,'Andreoni, Wanda, and Alessandro Curioni. \'New advances in chemistry and materials science with CPMD and parallel computing.\' Parallel Computing 26.7 (2000): 819-842.'),(55,NULL,NULL,NULL,'http://www.scm.com/'),(56,NULL,NULL,NULL,'John P. Perdew, Adrienn Ruzsinszky, GÃ¡bor I. Csonka, Lucian A. Constantin, and Jianwei Sun, Erratum: Workhorse Semilocal Density Functional for Condensed Matter Physics and Quantum Chemistry\' [Phys. Rev. Lett. 103, 026403 (2009)] Phys. Rev. Lett. 106 (2011) 179902(E).'),(57,NULL,NULL,NULL,'J. P. Perdew, J. A. Chevary, S. H. Vosko, K. A. Jackson, M. R. Pederson, D. J. Singh, and C. Fiolhais, \'Atoms, molecules, solids, and surfaces: Applications of the generalized gradient approximation for exchange and correlation,\' Phys. Rev. B, 46 (1992) 6671-87.'),(58,NULL,NULL,NULL,'M. Ernzerhof and J. P. Perdew, \'Generalized gradient approximation to the angle- and system-averaged exchange hole,\' J. Chem. Phys., 109 (1998)'),(59,NULL,NULL,NULL,'J. P. Perdew, K. Burke, and M. Ernzerhof, \'Generalized gradient approximation made simple,\' Phys. Rev. Lett., 77 (1996) 3865-68.'),(60,NULL,NULL,NULL,'Paci E, Lindorff-Larsen K, Dobson CM, Karplus M, Vendruscolo M, \'Transition state contact orders correlate with protein folding rates\'. J Mol Biol. 2005 Sep 23; 352(3):495-500.'),(61,NULL,NULL,NULL,'C. Adamo and V. Barone, \'Exchange functionals with improved long-range behavior and adiabatic connection methods without adjustable parameters: The mPW and mPW1PW models,\' J. Chem. Phys., 108 (1998) 664-75.'),(62,NULL,NULL,NULL,'Y. Zhao and D. G. Truhlar, \'A new local density functional for main-group thermochemistry, transition metal bonding, thermochemical kinetics, and noncovalent interactions,\' J. Chem. Phys., 125 (2006) 194101.'),(63,NULL,NULL,NULL,'A. D. Boese and J. M. L. Martin, \'Development of Density Functionals for Thermochemical Kinetics,\' J. Chem. Phys., 121 (2004) 3405-16.'),(64,NULL,NULL,NULL,'F. A. Hamprecht, A. Cohen, D. J. Tozer, and N. C. Handy, \'Development and assessment of new exchange-correlation functionals,\' J. Chem. Phys., 109 (1998) 6264-71.'),(65,NULL,NULL,NULL,'P. J. Wilson, T. J. Bradley, and D. J. Tozer, \'Hybrid exchange-correlation functional determined from thermochemical data and ab initio potentials,\' J. Chem. Phys., 115 (2001) 9233-42.'),(66,NULL,NULL,NULL,'A. D. Becke, \'Density-functional thermochemistry. IV. A new dynamical correlation functional and implications for exact-exchange mixing,\' J. Chem. Phys., 104 (1996) 1040-46.'),(67,NULL,NULL,NULL,'http://www.vasp.at/'),(68,NULL,NULL,NULL,'http://nmr.cit.nih.gov/xplor-nih/'),(69,NULL,NULL,NULL,'http://dasher.wustl.edu/ffe/'),(70,NULL,NULL,NULL,'http://www.wavefun.com/products/spartan.html'),(71,NULL,NULL,NULL,'http://www.turbomole.com/'),(72,NULL,NULL,NULL,'http://www.tremolo-x.com/'),(73,NULL,NULL,NULL,'http://www.q-chem.com/'),(74,NULL,NULL,NULL,'http://www.nanotec.es/products/siesta.php'),(75,NULL,NULL,NULL,'http://www.quantum-espresso.org/'),(76,NULL,NULL,NULL,'Hess, B; Bekker H, Berendsen HJC, Fraaije JGEM (1997). \'LINCS: A Linear Constraint Solver for Molecular Simulations\'. Journal of Computational Chemistry 18 (12): 1463-1472.'),(77,NULL,NULL,NULL,'KrÃ¤utler, Vincent, Wilfred F. van Gunsteren, and Philippe H. HÃ¼nenberger. (2001). \'A Fast SHAKE Algorithm to Solve Distance Constraint Equations for Small Molecules in Molecular Dynamics Simulations\'. Journal of Computational Chemistry 22 (5): 501-508.'),(78,NULL,NULL,NULL,'Gonnet, Pedro (2007). \'P-SHAKE: A quadratically convergent SHAKE in O(n^2)\'. Journal of Computational Physics 220 (2): 740-750.'),(79,NULL,NULL,NULL,'Miyamoto, S; Kollman PA (1992). \'SETTLE: An Analytical Version of the SHAKE and RATTLE Algorithm for Rigid Water Models\'. Journal of Computational Chemistry 13 (8): 952-962.'),(80,NULL,NULL,NULL,'Ryckaert, J-P; Ciccotti G, Berendsen HJC (1977). \'Numerical Integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes\'. Journal of Computational Physics 23 (3): 327-341.'),(81,NULL,NULL,NULL,'Tao, Peng; Xiongwu Wu, Bernard R. Brooks (2012). \'Maintain rigid structures in Verlet based Cartesian molecular dynamics simulations\'. The Journal of Chemical Physics 137: 134110.'),(82,NULL,NULL,NULL,'Lee, Sang-Ho; Kim Palmo, Samuel Krimm (2005). \'WIGGLE: A new constrained molecular dynamics algorithm in Cartesian coordinates\'. Journal of Computational Physics 210: 171-182.'),(83,NULL,NULL,NULL,'Andersen, Hans C. (1983). \'RATTLE: A \'Velocity\' Version of the SHAKE Algorithm for Molecular Dynamics Calculations\'. Journal of Computational Physics 52: 24-34.'),(84,NULL,NULL,NULL,'Guest, Martyn F.; Bush, Ian J.; Van Dam, Huub J. J.; Sherwood, Paul; Thomas, Jens M. H.; Van Lenthe, Joop H.; Havenith, Remco W. A.; Kendrick, John (2005). \'The GAMESS-UK structure package: algorithms, developments and applications\'. Molecular Physics 103 (6-8): 719-747.'),(85,NULL,NULL,NULL,'Van Duin, Adri CT, et al. \'ReaxFF: a reactive force field for hydrocarbons.\' The Journal of Physical Chemistry A 105.41 (2001): 9396-9409.'),(86,NULL,NULL,NULL,'Derreumaux, Philippe. \'From polypeptide sequences to structures using Monte Carlo simulations and an optimized potential.\' The Journal of chemical physics 111 (1999): 2301.'),(87,NULL,NULL,NULL,'Marrink, Siewert J., et al. \'The MARTINI force field: coarse grained model for biomolecular simulations.\' The Journal of Physical Chemistry B 111.27 (2007): 7812-7824.'),(88,NULL,NULL,NULL,'R. Ditchfield, W.J. Hehre and J.A. Pople, J. Chem. Phys. 54, 724 (1971).'),(89,NULL,NULL,NULL,'Chebaro, Yassmine, Samuela Pasquali, and Philippe Derreumaux. \'The Coarse-Grained OPEP Force Field for Non-Amyloid and Amyloid Proteins.\' The Journal of Physical Chemistry B 116.30 (2012): 8741-8752.'),(90,NULL,NULL,NULL,'Maupetit, Julien, P. Tuffery, and Philippe Derreumaux. \'A coarse-grained protein force field for folding and structure prediction.\' Proteins: Structure, Function, and Bioinformatics 69.2 (2007): 394-408.'),(91,NULL,NULL,NULL,'van Gunsteren, W. F., Billeter, S. R., Eking, A. A., Hiinenberger, P. H., Kriiger, P., Mark, A. E., Scott, W. R. P. and Tironi, I. G., \'Biomolecular Simulation, The GROMOS96 Manual and User Guide\', vdf Hochschulverlag AG an der ETH Ziirich and BIOMOS b.v., Zurich, Groningen, 1996.'),(92,NULL,NULL,NULL,'Soares, T. A., HÃ¼nenberger, P. H., Kastenholz, M. A., KrÃ¤utler, V., Lenz, T., Lins, R.D., Oostenbrink, C., and van Gunsteren, W.F., An improved nucleic acid parameter set for the GROMOS force field, Journal of Computational Chemistry, 26(7), (2005), 725-737'),(93,NULL,NULL,NULL,'W.J. Hehre, R. Ditchfield, R.F. Stewart, J.A. Pople, J. Chem. Phys.  2769 (1970)'),(94,NULL,NULL,NULL,'W.J. Hehre, R.F. Stewart and J.A. Pople, J. Chem. Phys. 2657 (1969)'),(95,NULL,NULL,NULL,'W.J. Pietro, B.A. Levy, W.J. Hehre and R.F. Stewart, J. Am. Chem. Soc. 19, 2225 (1980)'),(96,NULL,NULL,NULL,'W.J. Pietro and W.J. Hehre, J. Comp. Chem. 4, 241 (1983).'),(97,NULL,NULL,NULL,'Oostenbrink C., Villa, A., Mark, A.E., and van Gunsteren, W., A biomolecular force field based on the free enthalpy of hydration and solvation: the GROMOS force-field parameter sets 53A5 and 53A6, Journal of Computational Chemistry, 25, (2004), 1656-1676'),(98,NULL,NULL,NULL,'Schuler, L.D., Daura, X., and van Gusteren, W.F., An improved GROMOS96 force field for aliphatic hydrocarbons in the condensed phase, Journal of Computational Chemistry 22(11), (2001), 1205-1218'),(99,NULL,NULL,NULL,'Schmid N., Eichenberger A., Choutko A., Riniker S., Winger M., Mark A. & van Gunsteren W., Definition and testing of the GROMOS force-field versions 54A7 and 54B7, European Biophysics Journal, 40(7), (2011), 843-856'),(100,NULL,NULL,NULL,'Vanommeslaeghe K, Hatcher E, Acharya C, Kundu S, Zhong S, Shim J, Darian E, Guvench O, Lopes P, Vorobyov I, Mackerell AD Jr (2009). \'CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields\'. J Comput Chem 31 (4): 671-90.'),(101,NULL,NULL,NULL,'MacKerell, Jr. AD, Banavali N, Foloppe N (2001). \'Development and current status of the CHARMM force field for nucleic acids\'. Biopolymers 56 (4): 257-265.'),(102,NULL,NULL,NULL,'Best, Robert B., et al. \'Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone Phi,Psi and Side-Chain Chi1 and Chi2 Dihedral Angles.\' Journal of chemical theory and computation 8.9 (2012): 3257-3273.'),(103,NULL,NULL,NULL,'Reiher, III WH (1985). \'Theoretical studies of hydrogen bonding\'. PhD Thesis at Harvard University.'),(104,NULL,NULL,NULL,'MacKerell, Jr. AD, et al. (1998). \'All-atom empirical potential for molecular modeling and dynamics studies of proteins\'. J Phys Chem B 102 (18): 3586-3616.'),(105,NULL,NULL,NULL,'Kollman (1996), Acc. Chem. Res. 29, 461-469'),(106,NULL,NULL,NULL,'Cornell et al. (1995), JACS 117, 5179-5197'),(107,NULL,NULL,NULL,'Hornak et. al (2006). Proteins 65, 712-725'),(108,NULL,NULL,NULL,'Wang, J.; Cieplak, P.; Kollman, P.A. How well does a restrained electrostatic potential (RESP) model perform in calculating conformational energies of organic and biological molecules? J. Comput. Chem., 2000, 21, 1049-1074.'),(109,NULL,NULL,NULL,'Sorin, Eric J., and Vijay S. Pande. \'Exploring the helix-coil transition via all-atom equilibrium ensemble simulations.\' Biophysical Journal 88.4 (2005): 2472-2493.'),(110,NULL,NULL,NULL,'Lindorff-Larsen, Kresten, et al. \'Improved side-chain torsion potentials for the Amber ff99SB protein force field.\' Proteins: Structure, Function, and Bioinformatics 78.8 (2010): 1950-1958.'),(111,NULL,NULL,NULL,'Wang, Z.-X.; Zhang, W.; Wu, C.; Lei, H.; Cieplak, P.; Duan, Y. Strike a Balance: Optimization of backbone torsion parameters of AMBER polarizable force field for simulations of proteins and peptides. J. Comput. Chem., 2006, 27, 781-790.'),(112,NULL,NULL,NULL,'Perez A, Marchan I, Svozil D, Sponer J, Cheatham TE III., Laughton CA, Orozco M. Refinement of the AMBER force field for nucleic acids: improving the description of alpha/gamma conformers. Biophys. J. 2007;92:3817-3829.'),(113,NULL,NULL,NULL,'Duan, Y.; Wu, C.; Chowdhury, S.; Lee, M.C.; Xiong, G.; Zhang, W.; Yang, R.; Cieplak, P.; Luo, R.; Lee, T. A point-charge force field for molecular mechanics simulations of proteins based on condensed-phase quantum mechanical calculations. J. Comput. Chem.,2003, 24, 1999-2012.'),(114,NULL,NULL,NULL,'Kevin J. Bowers, Edmond Chow, Huafeng Xu, Ron O. Dror, Michael P. Eastwood, Brent A. Gregersen, John L. Klepeis, Istvan Kolossvary, Mark A. Moraes, Federico D. Sacerdoti, John K. Salmon, Yibing Shan, and David E. Shaw, \'Scalable Algorithms for Molecular Dynamics Simulations on Commodity Clusters,\' Proceedings of the ACM/IEEE Conference on Supercomputing (SC06), Tampa, Florida, 2006, November 11-17'),(115,NULL,NULL,NULL,'http://www.msg.chem.iastate.edu/gamess/'),(116,NULL,NULL,NULL,'Kirschner, K.N.; Yongye, A.B.; Tschampel, S.M.; GonzÃ¡lez-OuteiriÃ±o, J.; Daniels, C.R.; Foley, B.L.; Woods, R.J. GLYCAM06: A generalizable biomolecular force field. Carbohydrates. J. Comput. Chem., 2008, 29, 622-655.'),(117,NULL,NULL,NULL,'Wang, J.; Wolf, R.M.; Caldwell, J.W.; Kollamn, P.A.; Case, D.A. Development and testing of a general Amber force field. J. Comput. Chem., 2004, 25, 1157-1174'),(118,NULL,NULL,NULL,'AMBER Tools 12 manual. Available at: http://ambermd.org/doc12/AmberTools12.pdf'),(119,NULL,NULL,NULL,'AMBER Tools 10 manual. Available at: http://ambermd.org/doc10/AmberTools.pdf'),(120,NULL,NULL,NULL,'Ponder, J.W.; Wu, C.; Ren, P.; Pande, V.S.; Chodera, J.D.; Schieders, M.J.; Haque, I.; Mobley, D.L.; Lambrecht, D.S.; DiStasio, R.A. Jr.; Head-Gordon, M.; Clark, G.N.I.; Johnson, M.E.; Head-Gordon, T. Current status of the AMOEBA polarizable force field. J. Phys. Chem. B, 2010, 114, 2549-2564.'),(121,NULL,1984,'J. Chem. Phys.','H.J.C. Berendsen et al., J. Chem. Phys. 81, 3684 (1984)'),(122,NULL,1980,'J. Chem. Phys.','H.C. Andersen, J. Chem. Phys. 72, 2384 (1980)'),(123,NULL,1981,'J. Appl. Phys','M. Parinello and A. Rahman, J. Appl. Phys 52, 7182 (1981)'),(124,NULL,1985,'Phys. Rev.','Hoover, Phys. Rev. A 31 (1985) 1695'),(125,NULL,1984,'J. Chem. Phys.','S. NosÃ©, J. Chem. Phys. 81 (1984) 511'),(126,NULL,1984,'Mol. Phys.','S. NosÃ©, Mol. Phys. 52 (1984) 255'),(127,NULL,2007,'J. Chem. Phys.','G. Bussi, D. Donadio and M. Parrinello, Canonical sampling through velocity-rescaling, J. Chem. Phys. 126, 014101 (2007)');
/*!40000 ALTER TABLE `CITATION` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `CONCEPT`
--

DROP TABLE IF EXISTS `CONCEPT`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `CONCEPT` (
  `CUI` varchar(50) NOT NULL,
  `STR` varchar(100) NOT NULL,
  `IS_REL` tinyint(1) NOT NULL,
  PRIMARY KEY (`CUI`),
  UNIQUE KEY `UQ_CONCEPT_CUI` (`CUI`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `CONCEPT`
--

LOCK TABLES `CONCEPT` WRITE;
/*!40000 ALTER TABLE `CONCEPT` DISABLE KEYS */;
INSERT INTO `CONCEPT` VALUES ('C40017','CCSDTQ',0),('C40016','Coupled-cluster singles, doubles, and triples',0),('C40019','Completely renormalized CCSD(T) method',0),('C40018','CCSD and perturbative connected triples',0),('C40013','Coupled-cluster singles and doubles',0),('C40012','Coupled-cluster doubles',0),('C40015','Linearized coupled-cluster singles and doubles',0),('C40014','LCCD',0),('C40010','Configuration interaction singles, doubles, and triples',0),('QM01110','Correlation-consistent basis set',0),('C40011','CISDTQ',0),('C11101','Andersen thermostat',0),('C40008','MP5',0),('C40007','Fourth-order Moeller-Plesset',0),('C11103','Langevin thermostat',0),('C40006','MP3',0),('C11102','Berendsen thermostat',0),('C40005','Second-order Moeller-Plesset',0),('C40004','SCF',0),('C40003','Restricted open-shell Hartree-Fock',0),('C40002','UHF',0),('C40001','HF',0),('MTH14002','Quantum mechanics/Molecular Mechanics',0),('MTH14001','Multi-level QM method',0),('MTH14000','Hybrid method',0),('C40009','Configuration interaction singles and doubles',0),('MTH10000','Computational method',0),('QM01120','Pople basis set',0),('C11104','V-rescale thermostat',0),('C11105','Nose thermostat',0),('C11106','Nose-Hoover thermostat',0),('C11107','Nose-Poincare thermostat',0),('QM01130','Polarisation-consistent basis set',0),('PRM00011','QM parameter set',0),('C17104','OPLS-2005',0),('C17200','TIP3P',0),('QM01140','ECP basis set',0),('C17102','OPLS-UA',0),('PRM00301','MD setting',0),('C17103','OPLS-2001',0),('C17207','Single Point Charge water model',0),('C17208','SPC flexible',0),('C17205','TIP5P',0),('C17206','TIP6P',0),('C17203','TIP4P/Ice',0),('C17101','OPLS-AA',0),('C17204','TIP4P/2005',0),('PRM00201','Method setting',0),('C17201','TIP4P',0),('C17202','TIP4P-Ew',0),('PRM00001','Parameter set',0),('C17209','SPC/E',0),('C17303','Allner 2012',0),('PRM00101','Setting',0),('C17302','Joung/Cheatham 2008',0),('C17301','Beglov/Roux 1994',0),('C17300','Aqvist 1990',0),('C40021','QCISD(T)',0),('C40020','QCISD',0),('C17304','Merz 2013',0),('C10061','aug-cc-pV(Q+d)Z',0),('C10060','aug-cc-pV(T+d)Z',0),('C10065','aug-cc-pwCVTZ',0),('C10064','aug-cc-pwCVDZ',0),('C10063','aug-cc-pV(6+d)Z',0),('C10062','aug-cc-pV(5+d)Z',0),('C10069','aug-pc-1',0),('C10068','aug-pc-0',0),('C10067','aug-cc-pwCV5Z',0),('QM01150','Ahlrichs basis set',0),('C10066','aug-cc-pwCVQZ',0),('TSK00101','QM calculation',0),('HW01200','GPU',0),('FS01110','File format',0),('C10070','aug-pc-2',0),('C10072','aug-pc-4',0),('C10071','aug-pc-3',0),('C10074','Ahlrichs Polarization',0),('HW01100','CPU',0),('C10073','Ahlrichs Coulomb fitting',0),('FF00010','AMBER force field',0),('C10076','Ahlrichs VDZ',0),('FF00011','GROMOS force field',0),('C10075','Ahlrichs pvDZ',0),('FF00012','CHARMM force field',0),('C10077','Ahlrichs VTZ',0),('SW01100','Operating system',0),('FF00014','OPEP',0),('FF00013','OPLS',0),('FF00016','Water model',0),('FF00015','Ion force field parameter set',0),('SW01200','Software package',0),('QM02000','Spin multiplicity',0),('C11002','Split-valence basis set',0),('C11001','Minimal basis set',0),('C11004','Real space basis set',0),('C11003','Plane-wave basis set',0),('C10047','aug-cc-pV8Z',0),('C10046','aug-cc-pV6Z',0),('C10045','aug-cc-pV5Z',0),('C10044','aug-cc-pVQZ',0),('C10043','aug-cc-pVTZ',0),('C10042','aug-cc-pVDZ',0),('C10041','cc-pV(6+d)Z',0),('C10040','c-pV(5+d)Z',0),('C10048','aug-cc-pV9Z',0),('C10049','aug-cc-pVDZ-RI',0),('QM01000','Basis set',0),('C10050','aug-cc-pVTZ-RI',0),('C10056','aug-cc-pCVQZ',0),('C10055','aug-cc-pCVTZ',0),('C10058','aug-cc-pCV6Z',0),('C10057','aug-cc-pCV5Z',0),('C10052','aug-cc-pV5Z-RI',0),('C10051','aug-cc-pVQZ-RI',0),('C10054','aug-cc-pCVDZ',0),('C10053','aug-cc-pV6Z-RI',0),('C10059','aug-cc-pV(D+d)Z',0),('FS01100','File',0),('MTH12200','Semi-empirical method restricted to Pi electrons',0),('C12021','Gradient',0),('QM04004','Kerker pseudopotential',0),('C12020','Geometry optimization',0),('C12023','Hessian',0),('QM04002','Hamann PP',0),('C12022','Heat of formation',0),('QM04003','Bachelet-Hamann-Schluter pseudopotential',0),('C12025','Hyperfine coupling constants',0),('C12024','High accuracy energies',0),('C12027','Ionization potentials',0),('C12026','Hyperfine spectra tensors',0),('C12028','Infrared spectra',0),('C12029','Intrisic Reaction Coordinate',0),('QM04001','Troullier-Martins PP',0),('QM04000','Pseudopotential',0),('MTH12000','Semi-empirical method',0),('C12030','Intrisic Reaction Coordinate maxixmum',0),('C12034','Molecular orbitals',0),('C12033','Minimum crossing',0),('C12032','Polarizabilities/hyperpolarizabilities',0),('C12031','Pre-resonance Raman spectra',0),('C12038','Nuclei forces',0),('C12037','NMR spin-spin coupling constants',0),('C12036','NMR shielding and chemical shifts',0),('C12035','Multipole moments',0),('C12039','Optical rotations',0),('SYS00101','Implicit solvent',0),('SYS00102','Explicit solvent',0),('SYS00103','In vaccum',0),('C12041','Raman spectra',0),('C12040','Radiative transition moment',0),('C12047','Thermochemical analysis',0),('C12046','Spin-orbit coupling',0),('C12049','Time-dependent polarizabilities',0),('MD00307','Beeman integrator',0),('MD00306','Leap-frog integrator',0),('C12043','Saddle',0),('C12042','Raman optical activity',0),('C12045','Single point energy',0),('C12044','Scan',0),('C14002','PME',0),('C14003','Particle-Particle Particle-Mesh',0),('C14001','Cutoff',0),('C14004','Reaction field',0),('SW00400','CHARMM',0),('C12052','Vibration-rotation coupling',0),('C12051','UV/Visible spectra',0),('C12050','Trace gradient extremal',0),('C12056','Wave function property',0),('C12055','Wave function stability',0),('C12054','Volume',0),('C12053','Vibrational circular dichroism',0),('QM05200','Pure functional',0),('MTH12100','Semi-empirical method restricted to all-valence electrons',0),('PRM00031','Minimization parameter set',0),('QM05100','Hybrid exchange-correlational functional',0),('QM05000','Exchange-correlational functional',0),('PRM00021','Molecular dynamics parameter set',0),('HW01000','Computer hardware component',0),('MD00400','Barostat',0),('MD00303','Adams-Bashforth integrator',0),('MD00302','Runge-Kutta integrator',0),('MD00305','Verlet velocity integrator',0),('QM05300','Exchange-only functional',0),('MD00304','Verlet integrator',0),('MD00301','Euler integrator',0),('MD00300','MM integrator',0),('C40363','SAOP',0),('C12002','Antiferromagnetic coupling',0),('TSK10002','Heating process',0),('C40362','S12h',0),('C12003','Atomic charges',0),('SW00300','AMBER',0),('C40361','S12g',0),('C12004','Conical intersection point',0),('C40360','RPBE',0),('C12005','Delta G of solvation',0),('C12001','Anharmonic vibrational corrections',0),('C12101','Spectroscopic property',0),('C12102','Thermodynamic property',0),('C12103','Electronic property',0),('C12104','Physical property',0),('C16001','Accelerated Molecular Dynamics',0),('C16002','Adaptive Force Bias',0),('SW00500','CP2K',0),('C16003','CAFES',0),('C40368','TPSSh',0),('C16004','Forward flux simulation',0),('C40369','VSXC',0),('C16005','Locally enhanced sampling',0),('C40366','tHCTH',0),('C16006','Markov chain model',0),('C12007','Dynamics',0),('TSK10003','Equilibration process',0),('C40367','tHCTHhyb',0),('C16007','Max-flux approach',0),('C12006','Dipole moment',0),('TSK10004','Production MD process',0),('C40364','SSB-D',0),('C16008','Metadynamics',0),('C12009','Electron affinities',0),('C40365','tau-HCTH',0),('C16009','Milestoning',0),('C12008','Direct Reaction Coordinate',0),('C40372','wB97XD',0),('C12015','Energy decomposition analysis',0),('C40371','wB97X',0),('C12016','FMO geometry optimization',0),('C40374','XAlpha',0),('C12013','Electrostatic potential-derived charges',0),('C40373','X3LYP',0),('C12014','Electronic transition band shape',0),('SYS10000','Molecular system',0),('C12011','Electronic circular dichroism',0),('C12012','Electrostatic potential',0),('C40370','wB97',0),('C12010','Electron density',0),('C16011','Neural networks',0),('C16012','Nudged elastic band',0),('C16010','Multi-state Bennett Acceptance Ratio',0),('C16015','Steered dynamics',0),('C16016','String method',0),('TSK10001','Minimization process',0),('C16013','Reference potential spatial warping algorithm',0),('C16014','Replica-Exchange molecular dynamics',0),('C16019','Transition path sampling',0),('TSK10000','Computational process',0),('C12019','Gamma',0),('C12018','Frequency',0),('C16017','Swarm-of-trajectories',0),('C12017','Free state FMO calculation',0),('C16018','Targeted molecular dynamics',0),('C40339','M06HF',0),('SW00119','Firefly',0),('C40334','LC-PBE',0),('OS00005','Linux',0),('PRM00650','DRISM',0),('SW00118','NWChem',0),('C40333','LC-wPBE',0),('OS00004','Unix',0),('SW00117','NAMD',0),('C40332','LB94',0),('OS00003','MINIX',0),('SW00116','MOPAC',0),('C40331','KT1',0),('OS00002','MS Windows',0),('C40338','M06',0),('OS00001','Unix-like operating system',0),('C40337','M05-2X',0),('C40336','M05',0),('C40335','LDA',0),('SW00110','Gaussian',0),('SW00111','GROMACS',0),('SW00114','Jaguar',0),('OS00008','BSD',0),('SW00115','LAMMPS',0),('C40330','HSEhPBE',0),('OS00009','FreeBSD',0),('SW00112','GROMOS',0),('OS00006','OS X',0),('OS00007','BSD-based operating system',0),('C40329','HSEh1PBE',0),('PRM00649','Extended RISM',0),('C40328','HSE2PBE',0),('PRM00647','ALPB approach',0),('PRM00648','RISM',0),('PRM00645','Onufriev, Bashford, Case GB model',0),('PRM00646','GBn',0),('PRM00644','GB HCT',0),('C40321','HCTH',0),('OS00014','Solaris',0),('C40320','DSD-PBEBP86',0),('OS00013','AIX',0),('C40323','HCTH147',0),('OS00016','Debian',0),('PRM00501','Minimization setting',0),('C40322','HCTH93',0),('OS00015','Red Hat',0),('C40325','HFB',0),('OS00010','NetBSD',0),('C40324','HCTH407',0),('C40327','HSE',0),('OS00012','HP-UX',0),('C40326','HFS',0),('OS00011','OpenBSD',0),('SW00100','Abalone',0),('OS00017','Ubuntu',0),('OS00018','Linux Mint',0),('OS00019','Fedora',0),('SW00600','CPMD',0),('SW00200','ADF',0),('PRM00632','Accessible surface area-based model',0),('PRM00633','PB model',0),('PRM00634','Generalized Born model',0),('PRM00635','Ad hoc fast solvation model',0),('SYS00300','Molecule charge',0),('SYS00201','Periodic boundary conditions',0),('OS00023','Windows 8',0),('SYS00202','Non-periodic boundary conditions',0),('C40359','RPA',0),('OS00022','Windows XP',0),('C40358','revTPSS-D',0),('OS00021','Windows 7',0),('C40357','revTPSS',0),('OS00020','OpenSUSE',0),('C40356','revPBE',0),('C40355','PWPB95-D3',0),('MD00500','Temperature control',0),('C40354','PW6B95',0),('SYS00200','Boundary conditions',0),('C40353','PW91',0),('OS00024','Windows Vista',0),('PRM00631','Hybrid implicit/explicit solvent',0),('C40351','PBE1PBE',0),('C40352','PBEh1PBE',0),('C40350','PBE',0),('MD00700','Collision frequency',0),('PRM00621','Solvent model',0),('C16022','Umbrella sampling',0),('C16021','Transition interface sampling',0),('C16020','Transition state ensembles',0),('C40347','O3LYP',0),('C40346','mPW3PBE',0),('C40349','optB88-vdW',0),('C40348','OLYP',0),('C40343','mPW1LYP',0),('C40342','mPW1k',0),('C40345','mPW1PW91',0),('C40344','mPW1PBE',0),('C40340','M06-L',0),('C40341','M06-2X',0),('C30019','Comma-separated value file',0),('MTH11300','Enhanced sampling method',0),('C30017','CHARMM output',0),('C30018','Chemical Markup Language',0),('C30015','CHARMM coordinates',0),('C30016','CHARMM input',0),('C30013','AMBER NetCDF trajectory',0),('C30014','BMP',0),('C30012','AMBER Binpos trajectory',0),('C30011','AMBER binary trajectory',0),('C17504','Coarse-grain force field',0),('C30010','AMBER ASCII trajectory',0),('C17503','Reactive force field',0),('C17502','Polarizable force field',0),('C17501','Classical force field',0),('MTH11200','Minimization method',0),('MD00604','Generalized ensemble',0),('C30006','AMBER MD output',0),('MD00603','Isothermal-isobaric ensemble',0),('C30007','AMBER parmtop',0),('C30008','AMBER restart',0),('C30002','ADF output',0),('C30003','ADF Tape41',0),('PRM00601','Molecular system setting',0),('C30004','AMBER library',0),('PRM00600','System setting',0),('C30005','AMBER MD input',0),('C30001','ADF input',0),('C40315','BLYP',0),('C40316','BMK',0),('C40313','BHandH',0),('C40314','BHandHLYP',0),('C40311','B971',0),('C40312','B972',0),('MD00100','Restraint',0),('C40310','B97-D',0),('C30037','GROMACS restart',0),('C30038','GROMACS system topology',0),('SYS01000','Molecular system component',0),('MD00200','Constraint',0),('C30035','GROMACS MD input',0),('C30036','GROMACS MD output',0),('C40319','DCD-BLYP',0),('C40317','BP86',0),('C30039','GROMACS trajectory',0),('C40318','CAM-B3LYP',0),('MD00601','Microcanonical ensemble',0),('C30030','GAUSSIAN restart',0),('MD00602','Canonical ensemble',0),('MD00600','Ensemble',0),('C30034','GROMACS index',0),('C30033','GROMACS include topology',0),('C30032','GROMACS gro',0),('C30031','GIF',0),('SW00130','HyperChem',0),('C40302','B1B95',0),('C40303','B2PLYP',0),('C40304','B3LYP',0),('C40305','B3LYP*',0),('SW00127','VASP',0),('SW00128','X-PLOR',0),('C40300','APBE',0),('C40301','B1LYP',0),('C30024','GAMESS-UK input',0),('C30025','GAMESS-UK output',0),('C30026','GAUSSIAN chk',0),('C30027','GAUSSIAN com',0),('C40306','B3LYP-D',0),('C30028','GAUSSIAN cube',0),('C40307','B3P86',0),('C30029','GAUSSIAN log',0),('C40308','B3PW91',0),('C40309','B98',0),('C30021','Fasta',0),('C30020','DCD trajectory',0),('C30023','GAMESS output',0),('C30022','GAMESS input',0),('SW00124','TINKER',0),('SW00123','Spartan',0),('SW00126','TURBOMOLE',0),('SW00125','Tremolo-X',0),('SW00120','Q-Chem',0),('SW00122','SIESTA',0),('SW00121','Quantum ESPRESSO',0),('CPE00002','Computing environment',0),('FS01000','File system',0),('C30053','NWChem topology',0),('C30054','Protein Data Bank',0),('C13001','Linear Constraint Solver',0),('C30055','PNG',0),('C13002','M-SHAKE',0),('C30056','PSF',0),('C13003','P-SHAKE',0),('C13004','SETTLE',0),('C10039','cc-pV(Q+d)Z',0),('C30050','NWChem input',0),('C13005','SHAKE',0),('C10038','cc-pV(T+d)Z',0),('C30051','NWChem output',0),('C13006','SHAPE',0),('C10037','cc-pV(D+d)Z',0),('C30052','NWChem restart',0),('C13008','WIGGLE',0),('C10035','cc-pCV5Z',0),('C13007','RATTLE',0),('C10036','cc-pCV6Z',0),('SYS04120','3-letter residue sequence',0),('C10033','cc-pCVTZ',0),('C10034','cc-pCVQZ',0),('C10031','cc-pV6Z-RI',0),('C30058','Q-Chem input',0),('C10032','cc-pCVDZ',0),('C30057','Ptraj script',0),('C10030','cc-pV5Z-RI',0),('C30059','Q-Chem output',0),('QM01100','Atomic basis set',0),('TSK00201','MD run',0),('C30044','JPEG',0),('MTH13000','Ab initio method',0),('C30045','Mol2',0),('C30042','Jaguar input',0),('C30043','Jaguar output',0),('MTH13004','Coupled-cluster',0),('C10027','cc-pVDZ-RI',0),('C30040','HTML',0),('MTH13003','CI',0),('C10026','cc-pV9Z',0),('C30041','HyperChem HIN',0),('MTH13002','MP',0),('C10029','cc-pVQZ-RI',0),('MTH13001','HF',0),('C10028','cc-pVTZ-RI',0),('MTH13007','Multi-reference',0),('C10022','cc-pVQZ',0),('SW00800','GAMESS-UK',0),('C10023','cc-pV5Z',0),('MTH13005','Quadratic configuration interaction',0),('C10024','cc-pV6Z',0),('MTH13006','DFT',0),('C10025','cc-pV8Z',0),('C30049','NAMD configuration',0),('C30048','MOPAC Output',0),('C10020','cc-pVDZ',0),('C30047','MOPAC Internal',0),('C10021','cc-pVTZ',0),('C30046','MOPAC Cartesian',0),('MTH11220','Conjugate gradient',0),('C10018','LANL2TZ',0),('C10017','LANL2DZdp ECP Polarization',0),('C10016','LANL2DZdp ECP',0),('C10015','LANL2DZ+2s2p2d2f',0),('MTH13010','AIMD',0),('C10019','LANL2TZ(f)',0),('C10010','LANL08',0),('SYS04100','Canonical residue sequence',0),('C10013','LANL2DZ ECP',0),('C10014','LANL2DZ+1d1f',0),('C10011','LANL08(f)',0),('C10012','LANL2-[6s4p4d2f]',0),('SYS11000','Molecular system box',0),('C11011','Andersen',0),('MTH11210','Steepest descent',0),('C11012','Berendsen',0),('C11013','Hoover',0),('C17250','ReaxFF',0),('C11014','Parinello-Rahman',0),('MTH13026','G4',0),('C10005','4-21G',0),('C30062','TIFF',0),('C17350','OPEP 1',0),('MTH13025','Gaussian-3 method',0),('C10004','3-21G',0),('C30063','TINKER XYZ',0),('MTH13024','G2',0),('C10007','6-21G',0),('C30060','SDF',0),('MTH13023','G1',0),('C10006','4-31G',0),('C30061','Text',0),('QM03000','Frozen core',0),('C17354','MARTINI',0),('MTH13022','Gaussian-n method',0),('C10009','6-311G',0),('TSK00001','Computational task',0),('MTH13021','Thermochemical recipe',0),('C10008','6-31G',0),('C17353','OPEP 4',0),('C30064','TURBOMOLE Coordinate',0),('C17352','OPEP 3',0),('MTH13020','Post-Hartree-Fock method',0),('C30065','eXtensible Markup Language',0),('C17351','OPEP 2',0),('C17151','GROMOS 43A1',0),('C17152','GROMOS 43B1',0),('SYS04200','Software-specific residue sequence',0),('MTH13029','CBS',0),('C10000','STO-2G',0),('C17155','GROMOS 45A4',0),('SYS04110','One-letter residue sequence',0),('C10001','STO-3G',0),('C17156','GROMOS 53A5',0),('MTH13027','T1',0),('C10002','STO-4G',0),('C17153','GROMOS 43A2',0),('MTH13028','ccCA',0),('C10003','STO-6G',0),('C17154','GROMOS 45A3',0),('C17159','GROMOS 54B7',0),('C17054','CHARMM General Force Field',0),('C17157','GROMOS 53A6',0),('C17158','GROMOS 54A7',0),('C17052','CHARMM 27',0),('C17053','CHARMM 36',0),('C17050','CHARMM 19',0),('C17051','CHARMM 22',0),('MTH00010','Number of iterations',0),('MTH13030','Weizmann-n',0),('MTH00011','Number of time steps',0),('MTH13031','W1',0),('MTH13032','W2',0),('MTH13033','W3',0),('MTH13034','W4',0),('MTH13035','NEVPT',0),('MTH13036','CASPT',0),('MTH11000','Empirical method',0),('C17002','AMBER FF96',0),('PRM00401','QM setting',0),('C17001','AMBER FF94',0),('C17004','AMBER FF99SB',0),('C17003','AMBER FF99',0),('C17006','AMBER FF99 phi',0),('C17005','AMBER FF99SB ILDN',0),('C17008','AMBER FF02',0),('C17007','AMBER FF99bsc0',0),('MD00900','Force field',0),('MTH11100','Classical molecular dynamics',0),('C17009','AMBER FF03',0),('SW00700','Desmond',0),('SW00900','GAMESS-US',0),('MTH00100','Descriptor',0),('TSK00301','Minimization task',0),('C40200','Pariser-Parr-Pople',0),('C40201','CNDO/2',0),('C40202','INDO',0),('C40203','NDDO',0),('C40204','MINDO',0),('C40205','MNDO',0),('C40206','AM1',0),('MTH11120','Coarse-grain molecular dynamics',0),('C40207','PM3',0),('C40104','MR-CCSD',0),('C40208','PM6',0),('C40209','RM1',0),('C40100','Multi-configuration SCF',0),('C40101','Complete active space SCF',0),('C40102','Restricted active space self-consistent field',0),('C40103','MR-CI(SD)',0),('MTH00020','Time step length',0),('MD00800','Electrostatics model',0),('C40212','ZINDO',0),('C17013','GLYCAM 06',0),('C40213','SINDO',0),('C17012','AMBER GAFF',0),('C40210','SAM1',0),('C17011','AMBER FF12SB',0),('C40211','Sparkle/AM1',0),('C17010','AMBER FF10',0),('C40214','PM3MM',0),('C40215','PDDG',0),('MTH11110','All-atom molecular dynamics',0),('C17015','Atomic Mutlipole Optimized Energetics for Biomolecular Applications',0),('C17014','AMBER GLYCAM 06 EP',0),('IS_A','is a',1),('ANL00001','Analysis method',0),('ANL00002','2D-RMSd',0),('ANL00003','Matrix diagonalization',0),('ANL00004','Eigenmode analysis',0),('ANL00005','Auto correlation',0),('ANL00006','Time cross correlation',0),('ANL00007','Atom triplet angle calculation',0),('ANL00008','B-factor',0),('ANL00009','Atom map',0),('ANL00010','Coordinate average',0),('ANL00011','Atom neighbors',0),('ANL00012','Structural check',0),('ANL00013','Structure clustering',0),('ANL00014','Structure clustering with dihedral binning',0),('ANL00015','Atom interaction count',0),('ANL00016','Native contact losses',0),('ANL00017','Correlation coefficient calculation',0),('ANL00018','Diffusion calculation',0),('ANL00019','Dihedral calculation',0),('ANL00020','Distance calculation',0),('ANL00021','Distance RMSD',0),('ANL00022','Density grid',0),('ANL00023','Hydrogen bond analysis',0),('ANL00024','Hydrogen bond interaction energies',0),('ANL00025','Base pair stacking energies',0),('ANL00026','J-coupling',0),('ANL00027','Connolly surface area',0),('ANL00028','Nucleic acid structure parameter',0),('ANL00029','Nucleic acid shear',0),('ANL00030','Nucleic acid stretch',0),('ANL00031','Nucleic acid stagger',0),('ANL00032','Nucleic acid buckle',0),('ANL00033','Nucleic acid propel',0),('ANL00034','Nucleic acid opening',0),('ANL00035','Nucleic acid shift',0),('ANL00036','Nucleic acid slide',0),('ANL00037','Nucleic acid rise',0),('ANL00038','Nucleic acid tilt',0),('ANL00039','Nucleic acid roll',0),('ANL00040','Nucleic acid twist',0),('ANL00041','Nucleic acid x-displacement',0),('ANL00042','Nucleic acid y-displacement',0),('ANL00043','Principal axis alignment',0),('ANL00044','Coordinate projection',0),('ANL00045','Pucker',0),('ANL00046','Gyration radius',0),('ANL00047','Radial distribution',0),('ANL00048','RMSd',0),('ANL00049','RMSd correlation',0),('ANL00050','Running average',0),('ANL00051','Secondary structure',0),('ANL00052','Secondary structure changes',0),('ANL00053','PCA',0),('ANL00054','NOE',0),('ANL00055','Water shell',0),('ANL00056','Coarse-Grained Elastic Mesoscopic Model Flexibility',0),('ANL00057','Coarse-Grained Worm-like Chain Model Flexibility',0),('ANL00058','Ramachandran Plot',0),('ANL00059','SAS',0),('ANL00060','LCPO SAS',0),('ANL00061','Shrake-Rupley SAS',0),('ANL00062','TM-score',0),('ANL00063','MM-PBSA',0),('ANL00064','MM-GBSA',0),('TSK01000','Analysis task',0),('HW01300','Hard disk',0);
/*!40000 ALTER TABLE `CONCEPT` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `CONCEPT_CITATION`
--

DROP TABLE IF EXISTS `CONCEPT_CITATION`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `CONCEPT_CITATION` (
  `CUI` varchar(50) NOT NULL,
  `CITATION_ID` int(11) NOT NULL,
  PRIMARY KEY (`CUI`,`CITATION_ID`),
  KEY `CITATION_ID` (`CITATION_ID`),
  KEY `CUI` (`CUI`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `CONCEPT_CITATION`
--

LOCK TABLES `CONCEPT_CITATION` WRITE;
/*!40000 ALTER TABLE `CONCEPT_CITATION` DISABLE KEYS */;
INSERT INTO `CONCEPT_CITATION` VALUES ('C10001',93),('C10001',94),('C10001',95),('C10001',96),('C10006',88),('C11011',122),('C11012',121),('C11014',123),('C11101',122),('C11102',121),('C11104',127),('C11106',124),('C11106',125),('C11106',126),('C13001',76),('C13002',77),('C13003',78),('C13004',79),('C13005',80),('C13006',81),('C13007',83),('C13008',82),('C16001',17),('C16002',18),('C16003',20),('C16004',22),('C16005',24),('C16006',26),('C16007',27),('C16008',28),('C16009',29),('C16012',31),('C16013',34),('C16014',35),('C16015',32),('C16016',33),('C16018',37),('C16019',36),('C16020',60),('C17001',106),('C17002',105),('C17003',108),('C17004',107),('C17005',110),('C17006',109),('C17007',112),('C17008',111),('C17009',113),('C17010',119),('C17011',118),('C17012',117),('C17013',116),('C17014',116),('C17015',120),('C17050',103),('C17051',104),('C17052',101),('C17053',102),('C17054',100),('C17101',1),('C17102',3),('C17103',1),('C17104',1),('C17151',91),('C17152',91),('C17153',91),('C17154',98),('C17155',92),('C17156',97),('C17157',97),('C17158',99),('C17159',99),('C17200',2),('C17201',2),('C17202',8),('C17203',6),('C17204',7),('C17205',2),('C17207',4),('C17208',5),('C17209',9),('C17250',85),('C17300',13),('C17301',12),('C17302',11),('C17303',10),('C17304',14),('C17350',86),('C17350',87),('C17352',90),('C17353',89),('C40302',66),('C40311',64),('C40312',65),('C40316',63),('C40321',52),('C40329',51),('C40336',45),('C40337',44),('C40338',43),('C40339',38),('C40340',62),('C40345',61),('C40350',59),('C40352',58),('C40353',57),('C40357',56),('C40366',25),('C40368',21),('C40369',23),('C40373',30),('FF00013',1),('SW00100',53),('SW00110',46),('SW00111',47),('SW00112',50),('SW00114',48),('SW00115',49),('SW00116',42),('SW00117',41),('SW00118',40),('SW00119',39),('SW00120',73),('SW00121',75),('SW00122',74),('SW00123',70),('SW00124',69),('SW00125',72),('SW00126',71),('SW00127',67),('SW00128',68),('SW00200',55),('SW00300',16),('SW00400',15),('SW00500',19),('SW00600',54),('SW00700',114),('SW00800',84),('SW00900',115);
/*!40000 ALTER TABLE `CONCEPT_CITATION` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `CONCEPT_MAPPING`
--

DROP TABLE IF EXISTS `CONCEPT_MAPPING`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `CONCEPT_MAPPING` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `CUI` varchar(50) NOT NULL,
  `ONTOLOGY` varchar(50) NOT NULL,
  `CONCEPT` varchar(50) NOT NULL,
  PRIMARY KEY (`ID`),
  KEY `CUI` (`CUI`),
  KEY `ONTOLOGY` (`ONTOLOGY`)
) ENGINE=MyISAM AUTO_INCREMENT=21 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `CONCEPT_MAPPING`
--

LOCK TABLES `CONCEPT_MAPPING` WRITE;
/*!40000 ALTER TABLE `CONCEPT_MAPPING` DISABLE KEYS */;
INSERT INTO `CONCEPT_MAPPING` VALUES (3,'TSK00001','OBI','OBI_0200000'),(4,'MTH10000','IAO','IAO_0000104'),(5,'PRM00101','BFO','BFO_0000031'),(6,'PRM00001','BFO','BFO_0000031'),(7,'MTH00100','BFO','BFO_0000019'),(8,'SYS10000','CHEBI','CHEBI_24431'),(9,'SYS01000','CHEBI','CHEBI_24431'),(10,'HW01000','BFO','BFO_0000030'),(11,'FS01100','IAO','IAO_0000310'),(12,'SW01200','CHEMINF','CHEMINF_000342'),(13,'CPE00002','BFO','BFO_0000030'),(14,'SW01100','IAO','IAO_0000010'),(15,'FS01110','IAO','IAO_0000098'),(16,'SYS04100','SO','SO_0000001'),(17,'SYS04200','SO','SO_0000001'),(18,'FS01000','IAO','IAO_0000030'),(19,'SYS11000','BFO','BFO_0000028'),(20,'TSK10000','OBI','OBI_0200000');
/*!40000 ALTER TABLE `CONCEPT_MAPPING` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `CONCEPT_SEMANTIC_TYPE`
--

DROP TABLE IF EXISTS `CONCEPT_SEMANTIC_TYPE`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `CONCEPT_SEMANTIC_TYPE` (
  `CUI` varchar(50) NOT NULL,
  `TID` int(11) NOT NULL,
  PRIMARY KEY (`CUI`,`TID`),
  KEY `CUI` (`CUI`),
  KEY `TID` (`TID`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `CONCEPT_SEMANTIC_TYPE`
--

LOCK TABLES `CONCEPT_SEMANTIC_TYPE` WRITE;
/*!40000 ALTER TABLE `CONCEPT_SEMANTIC_TYPE` DISABLE KEYS */;
/*!40000 ALTER TABLE `CONCEPT_SEMANTIC_TYPE` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `CONCEPT_TERM`
--

DROP TABLE IF EXISTS `CONCEPT_TERM`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `CONCEPT_TERM` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `CUI` varchar(50) NOT NULL,
  `LANG` varchar(50) NOT NULL,
  `STR` varchar(255) NOT NULL,
  `IS_PREF` tinyint(1) NOT NULL,
  `IS_ABBR` tinyint(1) NOT NULL,
  PRIMARY KEY (`ID`),
  UNIQUE KEY `UQ_SYNONYM_ID` (`ID`),
  KEY `CUI` (`CUI`)
) ENGINE=MyISAM AUTO_INCREMENT=961 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `CONCEPT_TERM`
--

LOCK TABLES `CONCEPT_TERM` WRITE;
/*!40000 ALTER TABLE `CONCEPT_TERM` DISABLE KEYS */;
INSERT INTO `CONCEPT_TERM` VALUES (1,'C40017','en','CCSDTQ',1,0),(2,'C40017','en','Coupled-cluster singles, doubles, triples, and quadruples',0,0),(3,'C40016','en','Coupled-cluster singles, doubles, and triples',1,0),(4,'C40016','en','CCSDT',0,0),(6,'C40019','en','Completely renormalized CCSD(T) method',1,0),(7,'C40019','en','CR-CCSD(T)',0,0),(8,'C40018','en','CCSD and perturbative connected triples',1,0),(9,'C40018','en','CCSD(T)',0,0),(10,'C40018','en','Coupled-cluster singles and doubles with perturbative connected triples',0,0),(11,'C40013','en','Coupled-cluster singles and doubles',1,0),(12,'C40013','en','CCSD',0,0),(13,'C40012','en','Coupled-cluster doubles',1,0),(14,'C40012','en','CCD',0,0),(15,'C40015','en','Linearized coupled-cluster singles and doubles',1,0),(16,'C40015','en','LCCSD',0,0),(17,'C40014','en','LCCD',1,0),(18,'C40014','en','Linearized coupled-cluster doubles',0,0),(19,'C40010','en','Configuration interaction singles, doubles, and triples',1,0),(20,'C40010','en','CISDT',0,0),(21,'QM01110','en','Correlation-consistent basis set',1,0),(22,'C40011','en','CISDTQ',1,0),(23,'C40011','en','Configuration interaction singles, doubles, triples, and quadruples',0,0),(24,'C11101','en','Andersen thermostat',1,0),(25,'C40008','en','MP5',1,0),(26,'C40008','en','5th-order Moeller-Plesset',0,0),(27,'C40008','en','Fifth-order Moeller-Plesset',0,0),(28,'C40007','en','Fourth-order Moeller-Plesset',1,0),(29,'C40007','en','4th-order Moeller-Plesset',0,0),(30,'C40007','en','MP4',0,0),(31,'C11103','en','Langevin thermostat',1,0),(32,'C40006','en','MP3',1,0),(33,'C40006','en','Third-order Moeller-Plesset',0,0),(34,'C40006','en','3rd-order Moeller-Plesset',0,0),(35,'C11102','en','Berendsen thermostat',1,0),(36,'C40005','en','Second-order Moeller-Plesset',1,0),(37,'C40005','en','MP2',0,0),(38,'C40005','en','2nd-order Moeller-Plesset',0,0),(39,'C40004','en','SCF',1,0),(40,'C40004','en','Self-consistent field',0,0),(41,'C40003','en','Restricted open-shell Hartree-Fock',1,0),(42,'C40003','en','ROHF',0,0),(43,'C40002','en','UHF',1,0),(44,'C40002','en','Unrestricted Hartree-Fock',0,0),(45,'C40001','en','HF',1,0),(46,'C40001','en','Hartree-Fock',0,0),(47,'MTH14002','en','Quantum mechanics/Molecular Mechanics',1,0),(48,'MTH14002','en','Hybrid quantum mechanics/Molecular Mechanics',0,0),(49,'MTH14002','en','QM/MM',0,0),(50,'MTH14001','en','Multi-level QM method',1,0),(51,'MTH14001','en','Multi-level QM',0,0),(52,'MTH14001','en','Multi-level quantum mechanics',0,0),(53,'MTH14000','en','Hybrid method',1,0),(54,'C40009','en','Configuration interaction singles and doubles',1,0),(55,'C40009','en','CISD',0,0),(56,'MTH10000','en','Computational method',1,0),(57,'QM01120','en','Pople basis set',1,0),(58,'C11104','en','V-rescale thermostat',1,0),(59,'C11105','en','Nose thermostat',1,0),(60,'C11106','en','Nose-Hoover thermostat',1,0),(61,'C11107','en','Nose-Poincare thermostat',1,0),(62,'QM01130','en','Polarisation-consistent basis set',1,0),(63,'PRM00011','en','QM parameter set',1,0),(64,'PRM00011','en','Quantum chemistry parameter set',0,0),(65,'C17104','en','OPLS-2005',1,0),(66,'C17200','en','TIP3P',1,0),(67,'QM01140','en','ECP basis set',1,0),(68,'QM01140','en','Effective core potential basis set',0,0),(69,'C17102','en','OPLS-UA',1,0),(70,'PRM00301','en','MD setting',1,0),(71,'PRM00301','en','Molecular dynamics setting',0,0),(72,'C17103','en','OPLS-2001',1,0),(73,'C17207','en','Single Point Charge water model',1,0),(74,'C17207','en','SPC',0,0),(75,'C17208','en','SPC flexible',1,0),(76,'C17205','en','TIP5P',1,0),(77,'C17206','en','TIP6P',1,0),(78,'C17203','en','TIP4P/Ice',1,0),(79,'C17101','en','OPLS-AA',1,0),(80,'C17204','en','TIP4P/2005',1,0),(81,'PRM00201','en','Method setting',1,0),(82,'PRM00201','en','Computational method setting',0,0),(83,'C17201','en','TIP4P',1,0),(84,'C17202','en','TIP4P-Ew',1,0),(85,'PRM00001','en','Parameter set',1,0),(86,'PRM00001','en','Computational method parameter set',0,0),(87,'C17209','en','SPC/E',1,0),(88,'C17303','en','Allner 2012',1,0),(89,'PRM00101','en','Setting',1,0),(90,'C17302','en','Joung/Cheatham 2008',1,0),(91,'C17301','en','Beglov/Roux 1994',1,0),(92,'C17300','en','Aqvist 1990',1,0),(93,'C40021','en','QCISD(T)',1,0),(94,'C40021','en','QCISD and perturbative connected triples',0,0),(95,'C40021','en','Quadratic configuration interaction singles & doubles and perturbative connected triples',0,0),(96,'C40020','en','QCISD',1,0),(97,'C40020','en','Quadratic configuration interaction singles and doubles',0,0),(98,'C17304','en','Merz 2013',1,0),(99,'C10061','en','aug-cc-pV(Q+d)Z',1,0),(100,'C10060','en','aug-cc-pV(T+d)Z',1,0),(101,'C10065','en','aug-cc-pwCVTZ',1,0),(102,'C10064','en','aug-cc-pwCVDZ',1,0),(103,'C10063','en','aug-cc-pV(6+d)Z',1,0),(104,'C10062','en','aug-cc-pV(5+d)Z',1,0),(105,'C10069','en','aug-pc-1',1,0),(106,'C10068','en','aug-pc-0',1,0),(107,'C10067','en','aug-cc-pwCV5Z',1,0),(108,'QM01150','en','Ahlrichs basis set',1,0),(109,'C10066','en','aug-cc-pwCVQZ',1,0),(110,'TSK00101','en','QM calculation',1,0),(111,'TSK00101','en','QM task',0,0),(112,'TSK00101','en','Quantum chemistry calculation',0,0),(113,'TSK00101','en','Quantum calculation',0,0),(114,'HW01200','en','GPU',1,1),(115,'FS01110','en','File format',1,0),(117,'C10070','en','aug-pc-2',1,0),(118,'C10072','en','aug-pc-4',1,0),(119,'C10071','en','aug-pc-3',1,0),(120,'C10074','en','Ahlrichs Polarization',1,0),(121,'HW01100','en','CPU',1,1),(122,'C10073','en','Ahlrichs Coulomb fitting',1,0),(123,'FF00010','en','AMBER force field',1,0),(124,'FF00010','en','AMBER FF',0,0),(125,'C10076','en','Ahlrichs VDZ',1,0),(126,'FF00011','en','GROMOS force field',1,0),(127,'FF00011','en','GROMOS FF',0,0),(128,'C10075','en','Ahlrichs pvDZ',1,0),(129,'FF00012','en','CHARMM force field',1,0),(130,'FF00012','en','CHARMM FF',0,0),(131,'C10077','en','Ahlrichs VTZ',1,0),(132,'SW01100','en','Operating system',1,0),(133,'FF00014','en','OPEP',1,0),(134,'FF00013','en','OPLS',1,0),(135,'FF00013','en','Optimized Potential for Liquid Simulations',0,0),(136,'FF00016','en','Water model',1,0),(137,'FF00015','en','Ion force field parameter set',1,0),(138,'SW01200','en','Software package',1,0),(139,'QM02000','en','Spin multiplicity',1,0),(140,'C11002','en','Split-valence basis set',1,0),(141,'C11001','en','Minimal basis set',1,0),(142,'C11004','en','Real space basis set',1,0),(143,'C11003','en','Plane-wave basis set',1,0),(144,'C10047','en','aug-cc-pV8Z',1,0),(145,'C10046','en','aug-cc-pV6Z',1,0),(146,'C10045','en','aug-cc-pV5Z',1,0),(147,'C10044','en','aug-cc-pVQZ',1,0),(148,'C10043','en','aug-cc-pVTZ',1,0),(149,'C10042','en','aug-cc-pVDZ',1,0),(150,'C10041','en','cc-pV(6+d)Z',1,0),(151,'C10040','en','c-pV(5+d)Z',1,0),(152,'C10048','en','aug-cc-pV9Z',1,0),(153,'C10049','en','aug-cc-pVDZ-RI',1,0),(154,'QM01000','en','Basis set',1,0),(155,'C10050','en','aug-cc-pVTZ-RI',1,0),(156,'C10056','en','aug-cc-pCVQZ',1,0),(157,'C10055','en','aug-cc-pCVTZ',1,0),(158,'C10058','en','aug-cc-pCV6Z',1,0),(159,'C10057','en','aug-cc-pCV5Z',1,0),(160,'C10052','en','aug-cc-pV5Z-RI',1,0),(161,'C10051','en','aug-cc-pVQZ-RI',1,0),(162,'C10054','en','aug-cc-pCVDZ',1,0),(163,'C10053','en','aug-cc-pV6Z-RI',1,0),(164,'C10059','en','aug-cc-pV(D+d)Z',1,0),(165,'FS01100','en','File',1,0),(166,'MTH12200','en','Semi-empirical method restricted to Pi electrons',1,0),(167,'C12021','en','Gradient',1,0),(168,'QM04004','en','Kerker pseudopotential',1,0),(169,'C12020','en','Geometry optimization',1,0),(170,'C12023','en','Hessian',1,0),(171,'QM04002','en','Hamann PP',1,0),(172,'QM04002','en','Hamann-SchlÃ¼ter-Chiang pseudopotential',0,0),(173,'QM04002','en','Hamann pseudopotential',0,0),(174,'C12022','en','Heat of formation',1,0),(175,'QM04003','en','Bachelet-Hamann-Schluter pseudopotential',1,0),(176,'QM04003','en','Bachelet-Hamann-SchlÃ¼ter pseudopotential',0,0),(177,'C12025','en','Hyperfine coupling constants',1,0),(178,'C12024','en','High accuracy energies',1,0),(179,'C12027','en','Ionization potentials',1,0),(180,'C12026','en','Hyperfine spectra tensors',1,0),(181,'C12028','en','Infrared spectra',1,0),(182,'C12029','en','Intrisic Reaction Coordinate',1,0),(183,'QM04001','en','Troullier-Martins PP',1,0),(184,'QM04001','en','Troullier-Martins pseudopotential',0,0),(185,'QM04000','en','Pseudopotential',1,0),(186,'QM04000','en','Pseudo-potential',0,0),(187,'MTH12000','en','Semi-empirical method',1,0),(188,'C12030','en','Intrisic Reaction Coordinate maxixmum',1,0),(190,'C12034','en','Molecular orbitals',1,0),(191,'C12033','en','Minimum crossing',1,0),(192,'C12032','en','Polarizabilities/hyperpolarizabilities',1,0),(193,'C12031','en','Pre-resonance Raman spectra',1,0),(194,'C12038','en','Nuclei forces',1,0),(195,'C12037','en','NMR spin-spin coupling constants',1,0),(196,'C12036','en','NMR shielding and chemical shifts',1,0),(197,'C12035','en','Multipole moments',1,0),(198,'C12039','en','Optical rotations',1,0),(202,'SYS00101','en','Implicit solvent',1,0),(203,'SYS00101','en','Continuum solvent',0,0),(204,'SYS00102','en','Explicit solvent',1,0),(205,'SYS00103','en','In vaccum',1,0),(206,'SYS00103','en','In-vacuo',0,0),(207,'C12041','en','Raman spectra',1,0),(208,'C12040','en','Radiative transition moment',1,0),(209,'C12047','en','Thermochemical analysis',1,0),(210,'C12046','en','Spin-orbit coupling',1,0),(211,'C12049','en','Time-dependent polarizabilities',1,0),(212,'MD00307','en','Beeman integrator',1,0),(213,'MD00306','en','Leap-frog integrator',1,0),(214,'C12043','en','Saddle',1,0),(215,'C12042','en','Raman optical activity',1,0),(216,'C12045','en','Single point energy',1,0),(217,'C12044','en','Scan',1,0),(218,'C14002','en','PME',1,0),(219,'C14002','en','Particle Mesh Ewald',0,0),(220,'C14003','en','Particle-Particle Particle-Mesh',1,0),(221,'C14003','en','P3M',0,0),(222,'C14003','en','PPPM',0,0),(223,'C14001','en','Cutoff',1,0),(224,'C14004','en','Reaction field',1,0),(225,'SW00400','en','CHARMM',1,0),(226,'C12052','en','Vibration-rotation coupling',1,0),(228,'C12051','en','UV/Visible spectra',1,0),(229,'C12050','en','Trace gradient extremal',1,0),(230,'C12056','en','Wave function property',1,0),(231,'C12055','en','Wave function stability',1,0),(232,'C12054','en','Volume',1,0),(233,'C12053','en','Vibrational circular dichroism',1,0),(234,'QM05200','en','Pure functional',1,0),(235,'MTH12100','en','Semi-empirical method restricted to all-valence electrons',1,0),(236,'PRM00031','en','Minimization parameter set',1,0),(237,'QM05100','en','Hybrid exchange-correlational functional',1,0),(238,'QM05000','en','Exchange-correlational functional',1,0),(251,'PRM00021','en','Molecular dynamics parameter set',1,0),(252,'PRM00021','en','MD parameter set',0,0),(253,'HW01000','en','Hardware component',1,0),(254,'MD00400','en','Barostat',1,0),(255,'MD00400','en','Pressure control',0,0),(259,'MD00303','en','Adams-Bashforth integrator',1,0),(260,'MD00303','en','Adams-Bashforth-Moulton integrator',0,0),(264,'MD00302','en','Runge-Kutta integrator',1,0),(268,'MD00305','en','Verlet velocity integrator',1,0),(272,'QM05300','en','Exchange-only functional',1,0),(273,'MD00304','en','Verlet integrator',1,0),(274,'MD00301','en','Euler integrator',1,0),(278,'MD00300','en','MM integrator',1,0),(279,'MD00300','en','Molecular mechanics integrator',0,0),(280,'C40363','en','SAOP',1,0),(281,'C12002','en','Antiferromagnetic coupling',1,0),(956,'TSK10003','en','Equilibration process',1,0),(285,'C40362','en','S12h',1,0),(286,'C12003','en','Atomic charges',1,0),(287,'SW00300','en','AMBER',1,0),(958,'TSK10004','en','Production MD process',1,0),(957,'TSK10003','en','Equilibration',1,0),(291,'C40361','en','S12g',1,0),(292,'C12004','en','Conical intersection point',1,0),(959,'TSK10004','en','Production molecular dynamics process',1,0),(296,'C40360','en','RPBE',1,0),(297,'C12005','en','Delta G of solvation',1,0),(298,'C12001','en','Anharmonic vibrational corrections',1,0),(299,'C12101','en','Spectroscopic property',1,0),(300,'C12102','en','Thermodynamic property',1,0),(301,'C12103','en','Electronic property',1,0),(302,'C12104','en','Physical property',1,0),(303,'C16001','en','Accelerated Molecular Dynamics',1,0),(308,'C16002','en','Adaptive Force Bias',1,0),(309,'SW00500','en','CP2K',1,0),(313,'C16003','en','CAFES',1,0),(314,'C16003','en','Canonical adiabatic free energy sampling',0,0),(315,'C40368','en','TPSSh',1,0),(316,'C16004','en','Forward flux simulation',1,0),(320,'C40369','en','VSXC',1,0),(321,'C16005','en','Locally enhanced sampling',1,0),(325,'C40366','en','tHCTH',1,0),(326,'C16006','en','Markov chain model',1,0),(327,'C12007','en','Dynamics',1,0),(960,'TSK10004','en','Production molecular dynamics',1,0),(331,'C40367','en','tHCTHhyb',1,0),(332,'C16007','en','Max-flux approach',1,0),(333,'C12006','en','Dipole moment',1,0),(337,'C40364','en','SSB-D',1,0),(338,'C16008','en','Metadynamics',1,0),(339,'C12009','en','Electron affinities',1,0),(343,'C40365','en','tau-HCTH',1,0),(344,'C16009','en','Milestoning',1,0),(345,'C12008','en','Direct Reaction Coordinate',1,0),(346,'C40372','en','wB97XD',1,0),(347,'C12015','en','Energy decomposition analysis',1,0),(348,'C40371','en','wB97X',1,0),(349,'C12016','en','FMO geometry optimization',1,0),(350,'C40374','en','XAlpha',1,0),(351,'C12013','en','Electrostatic potential-derived charges',1,0),(352,'C40373','en','X3LYP',1,0),(353,'C12014','en','Electronic transition band shape',1,0),(354,'SYS10000','en','Molecular system',1,0),(355,'C12011','en','Electronic circular dichroism',1,0),(356,'C12012','en','Electrostatic potential',1,0),(357,'C40370','en','wB97',1,0),(358,'C12010','en','Electron density',1,0),(359,'C16011','en','Neural networks',1,0),(360,'C16012','en','Nudged elastic band',1,0),(361,'C16010','en','Multi-state Bennett Acceptance Ratio',1,0),(362,'C16010','en','MBAR',0,0),(955,'TSK10002','en','Heating process',1,0),(366,'C16015','en','Steered dynamics',1,0),(954,'TSK10001','en','Minimization',0,0),(369,'C16016','en','String method',1,0),(372,'C16013','en','Reference potential spatial warping algorithm',1,0),(953,'TSK10001','en','Minimization process',1,0),(375,'C16014','en','Replica-Exchange molecular dynamics',1,0),(952,'TSK10000','en','Computational process',1,0),(378,'C16019','en','Transition path sampling',1,0),(951,'SYS11000','en','Box',1,0),(381,'C12019','en','Gamma',1,0),(382,'C12018','en','Frequency',1,0),(383,'C16017','en','Swarm-of-trajectories',1,0),(384,'C12017','en','Free state FMO calculation',1,0),(385,'C16018','en','Targeted molecular dynamics',1,0),(386,'C40339','en','M06HF',1,0),(387,'SW00119','en','Firefly',1,0),(388,'C40334','en','LC-PBE',1,0),(389,'OS00005','en','Linux',1,0),(390,'PRM00650','en','DRISM',1,0),(391,'PRM00650','en','Dielectrically-consistent RISM',0,0),(392,'SW00118','en','NWChem',1,0),(393,'C40333','en','LC-wPBE',1,0),(394,'OS00004','en','Unix',1,0),(396,'SW00117','en','NAMD',1,0),(397,'C40332','en','LB94',1,0),(398,'OS00003','en','MINIX',1,0),(399,'SW00116','en','MOPAC',1,0),(400,'C40331','en','KT1',1,0),(401,'OS00002','en','MS Windows',1,0),(402,'OS00002','en','Windows',0,0),(403,'C40338','en','M06',1,0),(404,'OS00001','en','Unix-like operating system',1,0),(405,'C40337','en','M05-2X',1,0),(406,'C40336','en','M05',1,0),(407,'C40335','en','LDA',1,0),(409,'SW00110','en','Gaussian',1,0),(410,'SW00111','en','GROMACS',1,0),(411,'SW00114','en','Jaguar',1,0),(412,'OS00008','en','BSD',1,0),(413,'SW00115','en','LAMMPS',1,0),(414,'C40330','en','HSEhPBE',1,0),(415,'OS00009','en','FreeBSD',1,0),(416,'SW00112','en','GROMOS',1,0),(417,'OS00006','en','OS X',1,0),(418,'OS00007','en','BSD-based operating system',1,0),(419,'C40329','en','HSEh1PBE',1,0),(420,'PRM00649','en','Extended RISM',1,0),(421,'PRM00649','en','XRISM',0,0),(422,'C40328','en','HSE2PBE',1,0),(423,'PRM00647','en','ALPB approach',1,0),(424,'PRM00647','en','Analytical Linearized Poisson-Boltzmann',0,0),(425,'PRM00648','en','RISM',1,0),(426,'PRM00648','en','Reference Interaction Site Model',0,0),(427,'PRM00645','en','Onufriev, Bashford, Case GB model',1,0),(428,'PRM00645','en','GB OBC',0,0),(429,'PRM00646','en','GBn',1,0),(430,'PRM00646','en','Mongan, Simmerling, McCammon, Case, and Onufriev GB model',0,0),(431,'PRM00644','en','GB HCT',1,0),(432,'PRM00644','en','Hawkins, Cramer and Truhlar GB model',0,0),(433,'PRM00644','en','pairwise de-screening approach',0,0),(434,'C40321','en','HCTH',1,0),(435,'OS00014','en','Solaris',1,0),(436,'C40320','en','DSD-PBEBP86',1,0),(437,'OS00013','en','AIX',1,0),(438,'C40323','en','HCTH147',1,0),(439,'OS00016','en','Debian',1,0),(440,'PRM00501','en','Minimization setting',1,0),(441,'C40322','en','HCTH93',1,0),(442,'OS00015','en','Red Hat',1,0),(443,'C40325','en','HFB',1,0),(444,'OS00010','en','NetBSD',1,0),(445,'C40324','en','HCTH407',1,0),(446,'C40327','en','HSE',1,0),(447,'OS00012','en','HP-UX',1,0),(448,'C40326','en','HFS',1,0),(449,'OS00011','en','OpenBSD',1,0),(450,'SW00100','en','Abalone',1,0),(451,'OS00017','en','Ubuntu',1,0),(452,'OS00018','en','Linux Mint',1,0),(453,'OS00019','en','Fedora',1,0),(455,'SW00600','en','CPMD',1,0),(456,'SW00200','en','ADF',1,0),(457,'PRM00632','en','Accessible surface area-based model',1,0),(458,'PRM00633','en','PB model',1,0),(459,'PRM00633','en','Poisson-Boltzmann solvent',0,0),(460,'PRM00633','en','Poisson-Boltzmann model',0,0),(461,'PRM00634','en','Generalized Born model',1,0),(462,'PRM00634','en','GB model',0,0),(463,'PRM00635','en','Ad hoc fast solvation model',1,0),(464,'SYS00300','en','Molecule charge',1,0),(465,'SYS00300','en','Total charge',0,0),(466,'SYS00300','en','Formal charge',0,0),(467,'SYS00201','en','Periodic boundary conditions',1,0),(468,'OS00023','en','Windows 8',1,0),(469,'SYS00202','en','Non-periodic boundary conditions',1,0),(470,'C40359','en','RPA',1,0),(471,'OS00022','en','Windows XP',1,0),(472,'C40358','en','revTPSS-D',1,0),(473,'OS00021','en','Windows 7',1,0),(474,'C40357','en','revTPSS',1,0),(475,'OS00020','en','OpenSUSE',1,0),(476,'C40356','en','revPBE',1,0),(477,'C40355','en','PWPB95-D3',1,0),(478,'MD00500','en','Temperature control',1,0),(479,'MD00500','en','Thermostat',0,0),(480,'C40354','en','PW6B95',1,0),(481,'SYS00200','en','Boundary conditions',1,0),(482,'C40353','en','PW91',1,0),(483,'OS00024','en','Windows Vista',1,0),(484,'PRM00631','en','Hybrid implicit/explicit solvent',1,0),(485,'C40351','en','PBE1PBE',1,0),(486,'C40352','en','PBEh1PBE',1,0),(487,'C40350','en','PBE',1,0),(488,'MD00700','en','Collision frequency',1,0),(490,'PRM00621','en','Solvent model',1,0),(491,'C16022','en','Umbrella sampling',1,0),(492,'C16021','en','Transition interface sampling',1,0),(493,'C16020','en','Transition state ensembles',1,0),(494,'C40347','en','O3LYP',1,0),(495,'C40346','en','mPW3PBE',1,0),(496,'C40349','en','optB88-vdW',1,0),(497,'C40348','en','OLYP',1,0),(498,'C40343','en','mPW1LYP',1,0),(499,'C40342','en','mPW1k',1,0),(500,'C40345','en','mPW1PW91',1,0),(501,'C40344','en','mPW1PBE',1,0),(502,'C40340','en','M06-L',1,0),(503,'C40341','en','M06-2X',1,0),(504,'C30019','en','Comma-separated value file',1,0),(505,'C30019','en','CSV',0,0),(506,'MTH11300','en','Enhanced sampling method',1,0),(507,'C30017','en','CHARMM output',1,0),(508,'C30018','en','Chemical Markup Language',1,0),(509,'C30018','en','CML',0,0),(510,'C30015','en','CHARMM coordinates',1,0),(513,'C30016','en','CHARMM input',1,0),(514,'C30013','en','AMBER NetCDF trajectory',1,0),(515,'C30014','en','BMP',1,0),(516,'C30014','en','Bitmap',0,0),(517,'C30012','en','AMBER Binpos trajectory',1,0),(518,'C30012','en','Scripps Binpos',0,0),(519,'C30011','en','AMBER binary trajectory',1,0),(520,'C17504','en','Coarse-grain force field',1,0),(521,'C30010','en','AMBER ASCII trajectory',1,0),(522,'C17503','en','Reactive force field',1,0),(523,'C17502','en','Polarizable force field',1,0),(524,'C17501','en','Classical force field',1,0),(525,'MTH11200','en','Minimization method',1,0),(526,'MD00604','en','Generalized ensemble',1,0),(527,'C30006','en','AMBER MD output',1,0),(528,'MD00603','en','Isothermal-isobaric ensemble',1,0),(529,'MD00603','en','NPT',0,0),(530,'C30007','en','AMBER parmtop',1,0),(531,'C30007','en','AMBER topology',0,0),(532,'C30007','en','AMBER parameter',0,0),(533,'C30007','en','AMBER parameter/topology',0,0),(534,'C30008','en','AMBER restart',1,0),(535,'C30002','en','ADF output',1,0),(536,'C30003','en','ADF Tape41',1,0),(537,'PRM00601','en','Molecular system setting',1,0),(538,'C30004','en','AMBER library',1,0),(539,'PRM00600','en','System setting',1,0),(540,'C30005','en','AMBER MD input',1,0),(541,'C30001','en','ADF input',1,0),(543,'C40315','en','BLYP',1,0),(544,'C40316','en','BMK',1,0),(545,'C40313','en','BHandH',1,0),(546,'C40314','en','BHandHLYP',1,0),(547,'C40311','en','B971',1,0),(548,'C40312','en','B972',1,0),(549,'MD00100','en','Restraint',1,0),(550,'C40310','en','B97-D',1,0),(551,'C30037','en','GROMACS restart',1,0),(552,'C30038','en','GROMACS system topology',1,0),(553,'SYS01000','en','Molecular system component',1,0),(554,'MD00200','en','Constraint',1,0),(555,'C30035','en','GROMACS MD input',1,0),(556,'C30036','en','GROMACS MD output',1,0),(557,'C40319','en','DCD-BLYP',1,0),(558,'C40317','en','BP86',1,0),(559,'C30039','en','GROMACS trajectory',1,0),(560,'C40318','en','CAM-B3LYP',1,0),(561,'MD00601','en','Microcanonical ensemble',1,0),(562,'MD00601','en','NVE',0,0),(563,'C30030','en','GAUSSIAN restart',1,0),(564,'MD00602','en','Canonical ensemble',1,0),(565,'MD00602','en','NVT',0,0),(566,'MD00600','en','Ensemble',1,0),(567,'C30034','en','GROMACS index',1,0),(568,'C30033','en','GROMACS include topology',1,0),(569,'C30032','en','GROMACS gro',1,0),(570,'C30031','en','GIF',1,0),(571,'C30031','en','Graphics Interchange File',0,0),(572,'SW00130','en','HyperChem',1,0),(573,'C40302','en','B1B95',1,0),(574,'C40303','en','B2PLYP',1,0),(575,'C40304','en','B3LYP',1,0),(576,'C40305','en','B3LYP*',1,0),(577,'SW00127','en','VASP',1,0),(578,'SW00128','en','X-PLOR',1,0),(579,'SW00128','en','XPLOR',0,0),(580,'C40300','en','APBE',1,0),(581,'C40301','en','B1LYP',1,0),(582,'C30024','en','GAMESS-UK input',1,0),(583,'C30025','en','GAMESS-UK output',1,0),(584,'C30026','en','GAUSSIAN chk',1,0),(585,'C30027','en','GAUSSIAN com',1,0),(586,'C40306','en','B3LYP-D',1,0),(587,'C30028','en','GAUSSIAN cube',1,0),(588,'C40307','en','B3P86',1,0),(589,'C30029','en','GAUSSIAN log',1,0),(590,'C40308','en','B3PW91',1,0),(591,'C40309','en','B98',1,0),(592,'C30021','en','Fasta',1,0),(593,'C30020','en','DCD trajectory',1,0),(594,'C30023','en','GAMESS output',1,0),(595,'C30022','en','GAMESS input',1,0),(596,'SW00124','en','TINKER',1,0),(597,'SW00123','en','Spartan',1,0),(599,'SW00126','en','TURBOMOLE',1,0),(600,'SW00125','en','Tremolo-X',1,0),(601,'SW00120','en','Q-Chem',1,0),(602,'SW00120','en','Qchem',0,0),(603,'SW00122','en','SIESTA',1,0),(604,'SW00121','en','Quantum ESPRESSO',1,0),(606,'CPE00002','en','Computing environment',1,0),(607,'CPE00002','en','Computing platform',0,0),(608,'FS01000','en','File system',1,0),(609,'C30053','en','NWChem topology',1,0),(610,'C30054','en','Protein Data Bank',1,0),(611,'C30054','en','PDB',0,0),(612,'C13001','en','Linear Constraint Solver',1,0),(613,'C13001','en','LINCS',0,0),(614,'C30055','en','PNG',1,0),(615,'C30055','en','Portable Networks Image',0,0),(616,'C13002','en','M-SHAKE',1,0),(617,'C30056','en','PSF',1,0),(618,'C30056','en','Protein Structure File',0,0),(619,'C13003','en','P-SHAKE',1,0),(620,'C13004','en','SETTLE',1,0),(621,'C10039','en','cc-pV(Q+d)Z',1,0),(622,'C30050','en','NWChem input',1,0),(623,'C13005','en','SHAKE',1,0),(624,'C10038','en','cc-pV(T+d)Z',1,0),(625,'C30051','en','NWChem output',1,0),(627,'C13006','en','SHAPE',1,0),(628,'C10037','en','cc-pV(D+d)Z',1,0),(629,'C30052','en','NWChem restart',1,0),(630,'C13008','en','WIGGLE',1,0),(631,'C10035','en','cc-pCV5Z',1,0),(632,'C13007','en','RATTLE',1,0),(633,'C10036','en','cc-pCV6Z',1,0),(636,'SYS04120','en','3-letter residue sequence',1,0),(637,'SYS04120','en','Three-letter residue sequence',0,0),(638,'SYS04120','en','3-letter code sequence',0,0),(639,'C10033','en','cc-pCVTZ',1,0),(640,'C10034','en','cc-pCVQZ',1,0),(641,'C10031','en','cc-pV6Z-RI',1,0),(642,'C30058','en','Q-Chem input',1,0),(643,'C10032','en','cc-pCVDZ',1,0),(644,'C30057','en','Ptraj script',1,0),(645,'C10030','en','cc-pV5Z-RI',1,0),(646,'C30059','en','Q-Chem output',1,0),(647,'QM01100','en','Atomic basis set',1,0),(648,'TSK00201','en','MD run',1,0),(649,'TSK00201','en','MD task',0,0),(650,'TSK00201','en','Molecular dynamics run',0,0),(651,'C30044','en','JPEG',1,0),(652,'C30044','en','Joint Photographic Experts Group',0,0),(653,'MTH13000','en','Ab initio method',1,0),(654,'C30045','en','Mol2',1,0),(655,'C30042','en','Jaguar input',1,0),(656,'C30043','en','Jaguar output',1,0),(657,'MTH13004','en','Coupled-cluster',1,0),(658,'MTH13004','en','CC',0,0),(659,'C10027','en','cc-pVDZ-RI',1,0),(660,'C30040','en','HTML',1,0),(661,'C30040','en','HyperText Markup Language',0,0),(662,'MTH13003','en','CI',1,0),(663,'MTH13003','en','Configuration interaction',0,0),(664,'C10026','en','cc-pV9Z',1,0),(665,'C30041','en','HyperChem HIN',1,0),(666,'MTH13002','en','MP',1,0),(667,'MTH13002','en','Moeller-Plesset',0,0),(668,'MTH13002','en','Perturbation theory',0,0),(669,'MTH13002','en','Moller-Plesset',0,0),(670,'C10029','en','cc-pVQZ-RI',1,0),(671,'MTH13001','en','HF',1,0),(672,'MTH13001','en','Hartree-Fock',0,0),(673,'C10028','en','cc-pVTZ-RI',1,0),(674,'MTH13007','en','Multi-reference',1,0),(675,'MTH13007','en','MR',0,0),(676,'C10022','en','cc-pVQZ',1,0),(677,'SW00800','en','GAMESS-UK',1,0),(678,'C10023','en','cc-pV5Z',1,0),(679,'MTH13005','en','Quadratic configuration interaction',1,0),(680,'MTH13005','en','QCI',0,0),(681,'C10024','en','cc-pV6Z',1,0),(682,'MTH13006','en','DFT',1,0),(683,'MTH13006','en','Density functional theory',0,0),(684,'C10025','en','cc-pV8Z',1,0),(685,'C30049','en','NAMD configuration',1,0),(686,'C30048','en','MOPAC Output',1,0),(687,'C10020','en','cc-pVDZ',1,0),(688,'C30047','en','MOPAC Internal',1,0),(689,'C10021','en','cc-pVTZ',1,0),(690,'C30046','en','MOPAC Cartesian',1,0),(691,'MTH11220','en','Conjugate gradient',1,0),(692,'C10018','en','LANL2TZ',1,0),(693,'C10017','en','LANL2DZdp ECP Polarization',1,0),(694,'C10016','en','LANL2DZdp ECP',1,0),(695,'C10015','en','LANL2DZ+2s2p2d2f',1,0),(696,'MTH13010','en','AIMD',1,0),(697,'MTH13010','en','Ab-initio MD',0,0),(698,'MTH13010','en','Ab-initio molecular dynamics',0,0),(699,'C10019','en','LANL2TZ(f)',1,0),(701,'C10010','en','LANL08',1,0),(702,'SYS04100','en','Canonical residue sequence',1,0),(703,'SYS04100','en','Normalized residue sequence',0,0),(704,'SYS04100','en','Canonical sequence',0,0),(705,'SYS04100','en','Normalized sequence',0,0),(706,'C10013','en','LANL2DZ ECP',1,0),(707,'C10014','en','LANL2DZ+1d1f',1,0),(708,'C10011','en','LANL08(f)',1,0),(709,'C10012','en','LANL2-[6s4p4d2f]',1,0),(950,'SYS11000','en','Molecular system box',0,0),(711,'C11011','en','Andersen',1,0),(712,'MTH11210','en','Steepest descent',1,0),(713,'C11012','en','Berendsen',1,0),(714,'C11013','en','Hoover',1,0),(715,'C17250','en','ReaxFF',1,0),(716,'C11014','en','Parinello-Rahman',1,0),(717,'MTH13026','en','G4',1,0),(718,'MTH13026','en','Gaussian-4 method',0,0),(719,'C10005','en','4-21G',1,0),(720,'C30062','en','TIFF',1,0),(721,'C17350','en','OPEP 1',1,0),(722,'MTH13025','en','Gaussian-3 method',1,0),(723,'MTH13025','en','G3',0,0),(724,'C10004','en','3-21G',1,0),(725,'C30063','en','TINKER XYZ',1,0),(726,'MTH13024','en','G2',1,0),(727,'MTH13024','en','Gaussian-2 method',0,0),(728,'C10007','en','6-21G',1,0),(729,'C30060','en','SDF',1,0),(730,'C30060','en','MDL SDF',0,0),(731,'C30060','en','MDL Structure Data Format',0,0),(732,'MTH13023','en','G1',1,0),(733,'MTH13023','en','Gaussian-1 method',0,0),(734,'C10006','en','4-31G',1,0),(735,'C30061','en','Text',1,0),(736,'QM03000','en','Frozen core',1,0),(737,'C17354','en','MARTINI',1,0),(738,'MTH13022','en','Gaussian-n method',1,0),(739,'C10009','en','6-311G',1,0),(740,'TSK00001','en','Computational task',1,0),(741,'MTH13021','en','Thermochemical recipe',1,0),(742,'MTH13021','en','Quantum chemistry composite method',0,0),(743,'C10008','en','6-31G',1,0),(744,'C17353','en','OPEP 4',1,0),(745,'C30064','en','TURBOMOLE Coordinate',1,0),(746,'C17352','en','OPEP 3',1,0),(747,'MTH13020','en','Post-Hartree-Fock method',1,0),(748,'MTH13020','en','Post-HF method',0,0),(749,'C30065','en','eXtensible Markup Language',1,0),(750,'C30065','en','XML',0,0),(751,'C17351','en','OPEP 2',1,0),(752,'C17151','en','GROMOS 43A1',1,0),(753,'C17152','en','GROMOS 43B1',1,0),(756,'SYS04200','en','Software-specific residue sequence',1,0),(757,'SYS04200','en','Software-specific sequence',0,0),(758,'MTH13029','en','CBS',1,0),(759,'MTH13029','en','Complete basis set method',0,0),(760,'C10000','en','STO-2G',1,0),(761,'C17155','en','GROMOS 45A4',1,0),(762,'SYS04110','en','One-letter residue sequence',1,0),(763,'SYS04110','en','1-letter code sequence',0,0),(764,'SYS04110','en','1-letter residue sequence',0,0),(765,'C10001','en','STO-3G',1,0),(766,'C17156','en','GROMOS 53A5',1,0),(767,'MTH13027','en','T1',1,0),(768,'C10002','en','STO-4G',1,0),(769,'C17153','en','GROMOS 43A2',1,0),(770,'MTH13028','en','ccCA',1,0),(771,'MTH13028','en','Correlation consistent Composite Approach',0,0),(772,'C10003','en','STO-6G',1,0),(773,'C17154','en','GROMOS 45A3',1,0),(774,'C17159','en','GROMOS 54B7',1,0),(775,'C17054','en','CHARMM General Force Field',1,0),(776,'C17054','en','CgenFF',0,0),(777,'C17054','en','CHARMM CGenFF',0,0),(778,'C17157','en','GROMOS 53A6',1,0),(779,'C17158','en','GROMOS 54A7',1,0),(780,'C17052','en','CHARMM 27',1,0),(781,'C17053','en','CHARMM 36',1,0),(782,'C17050','en','CHARMM 19',1,0),(783,'C17051','en','CHARMM 22',1,0),(784,'MTH00010','en','Number of iterations',1,0),(785,'MTH13030','en','Weizmann-n',1,0),(786,'MTH13030','en','Weizmann-n method',0,0),(787,'MTH00011','en','Number of time steps',1,0),(788,'MTH13031','en','W1',1,0),(789,'MTH13031','en','Weizmann-1',0,0),(790,'MTH13032','en','W2',1,0),(791,'MTH13032','en','Weizmann-2',0,0),(792,'MTH13033','en','W3',1,0),(793,'MTH13033','en','Weizmann-3',0,0),(794,'MTH13034','en','W4',1,0),(795,'MTH13034','en','Weizmann-4',0,0),(796,'MTH13035','en','NEVPT',1,0),(797,'MTH13035','en','n-electron valence state perturbation',0,0),(798,'MTH13036','en','CASPT',1,0),(799,'MTH13036','en','CASPTn',0,0),(800,'MTH13036','en','Complete active space perturbation',0,0),(801,'MTH11000','en','Empirical method',1,0),(802,'C17002','en','AMBER FF96',1,0),(803,'PRM00401','en','QM setting',1,0),(804,'PRM00401','en','Quantum chemistry setting',0,0),(805,'C17001','en','AMBER FF94',1,0),(806,'C17004','en','AMBER FF99SB',1,0),(807,'C17003','en','AMBER FF99',1,0),(808,'C17006','en','AMBER FF99 phi',1,0),(809,'C17005','en','AMBER FF99SB ILDN',1,0),(810,'C17008','en','AMBER FF02',1,0),(811,'C17007','en','AMBER FF99bsc0',1,0),(812,'MD00900','en','Force field',1,0),(813,'MTH11100','en','Classical molecular dynamics',1,0),(814,'C17009','en','AMBER FF03',1,0),(815,'SW00700','en','Desmond',1,0),(816,'SW00900','en','GAMESS-US',1,0),(817,'MTH00100','en','Descriptor',1,0),(818,'MTH00100','en','Property',0,0),(819,'TSK00301','en','Minimization task',1,0),(820,'TSK00301','en','Minimization run',0,0),(821,'TSK00301','en','Minimization',0,0),(823,'C40200','en','Pariser-Parr-Pople',1,0),(824,'C40200','en','PPP',0,0),(825,'C40201','en','CNDO/2',1,0),(826,'C40202','en','INDO',1,0),(827,'C40203','en','NDDO',1,0),(828,'C40204','en','MINDO',1,0),(829,'C40205','en','MNDO',1,0),(830,'C40206','en','AM1',1,0),(831,'MTH11120','en','Coarse-grain molecular dynamics',1,0),(832,'C40207','en','PM3',1,0),(833,'C40104','en','MR-CCSD',1,0),(834,'C40104','en','Multi-reference Coupled-cluster',0,0),(835,'C40104','en','Multi-reference CI',0,0),(836,'C40208','en','PM6',1,0),(837,'C40209','en','RM1',1,0),(838,'C40100','en','Multi-configuration SCF',1,0),(839,'C40100','en','Multi-configuration self-consistent field',0,0),(840,'C40100','en','MC-SCF',0,0),(841,'C40101','en','Complete active space SCF',1,0),(842,'C40101','en','CASSCF',0,0),(843,'C40101','en','Complete active space self-consistent field',0,0),(844,'C40102','en','Restricted active space self-consistent field',1,0),(845,'C40102','en','Restricted active space SCF',0,0),(846,'C40102','en','RASSCF',0,0),(847,'C40103','en','MR-CI(SD)',1,0),(848,'C40103','en','Multi-reference Configuration Interaction',0,0),(849,'MTH00020','en','Time step length',1,0),(850,'MTH00020','en','Delta t',0,0),(851,'MD00800','en','Electrostatics model',1,0),(852,'C40212','en','ZINDO',1,0),(853,'C17013','en','GLYCAM 06',1,0),(854,'C17013','en','AMBER GLYCAM 06',0,0),(855,'C40213','en','SINDO',1,0),(856,'C17012','en','AMBER GAFF',1,0),(857,'C17012','en','General Amber Force Field',0,0),(858,'C17012','en','GAFF',0,0),(859,'C40210','en','SAM1',1,0),(860,'C17011','en','AMBER FF12SB',1,0),(861,'C40211','en','Sparkle/AM1',1,0),(862,'C17010','en','AMBER FF10',1,0),(863,'C40214','en','PM3MM',1,0),(864,'C40215','en','PDDG',1,0),(865,'MTH11110','en','All-atom molecular dynamics',1,0),(866,'C17015','en','Atomic Mutlipole Optimized Energetics for Biomolecular Applications',1,0),(867,'C17015','en','AMOEBA',0,0),(868,'C17015','en','AMBER AMOEBA',0,0),(869,'C17014','en','AMBER GLYCAM 06 EP',1,0),(870,'C17014','en','GLYCAM 06 EP',0,0),(871,'ANL00002','en','2D-RMSd',1,0),(872,'ANL00003','en','Matrix diagonalization',1,0),(873,'ANL00004','en','Eigenmode analysis',1,0),(874,'ANL00005','en','Auto correlation',1,0),(875,'ANL00006','en','Time cross correlation',1,0),(876,'ANL00007','en','Atom triplet angle calculation',1,0),(877,'ANL00008','en','B-factor',1,0),(878,'ANL00009','en','Atom map',1,0),(879,'ANL00010','en','Coordinate average',1,0),(880,'ANL00011','en','Atom neighbors',1,0),(881,'ANL00012','en','Structural check',1,0),(882,'ANL00013','en','Structure clustering',1,0),(883,'ANL00014','en','Structure clustering with dihedral binning',1,0),(884,'ANL00015','en','Atom interaction count',1,0),(885,'ANL00016','en','Native contact losses',1,0),(886,'ANL00017','en','Correlation coefficient calculation',1,0),(887,'ANL00018','en','Diffusion calculation',1,0),(888,'ANL00019','en','Dihedral calculation',1,0),(889,'ANL00020','en','Distance calculation',1,0),(890,'ANL00021','en','Distance RMSD',1,0),(891,'ANL00022','en','Density grid',1,0),(892,'ANL00023','en','Hydrogen bond analysis',1,0),(893,'ANL00024','en','Hydrogen bond interaction energies',1,0),(894,'ANL00025','en','Base pair stacking energies',1,0),(895,'ANL00026','en','J-coupling',1,0),(896,'ANL00027','en','Connolly surface area',1,0),(897,'ANL00028','en','Nucleic acid structure parameters',1,0),(898,'ANL00029','en','Nucleic acid shear',1,0),(899,'ANL00030','en','Nucleic acid stretch',1,0),(900,'ANL00031','en','Nucleic acid stagger',1,0),(901,'ANL00032','en','Nucleic acid buckle',1,0),(902,'ANL00033','en','Nucleic acid propel',1,0),(903,'ANL00034','en','Nucleic acid opening',1,0),(904,'ANL00035','en','Nucleic acid shift',1,0),(905,'ANL00036','en','Nucleic acid slide',1,0),(906,'ANL00037','en','Nucleic acid rise',1,0),(907,'ANL00038','en','Nucleic acid tilt',1,0),(908,'ANL00039','en','Nucleic acid roll',1,0),(909,'ANL00040','en','Nucleic acid twist',1,0),(910,'ANL00041','en','Nucleic acid x-displacement',1,0),(911,'ANL00042','en','Nucleic acid y-displacement',1,0),(912,'ANL00043','en','Principal axis alignment',1,0),(913,'ANL00044','en','Coordinate projection',1,0),(914,'ANL00045','en','Pucker',1,0),(915,'ANL00046','en','Gyration radius',1,0),(916,'ANL00047','en','Radial distribution',1,0),(917,'ANL00048','en','RMSd',1,1),(918,'ANL00048','en','Root mean square deviation',1,0),(919,'ANL00049','en','RMSd correlation',1,0),(920,'ANL00050','en','Running average',1,0),(921,'ANL00051','en','Secondary structure',1,0),(922,'ANL00052','en','Secondary structure changes',1,0),(923,'ANL00053','en','PCA',0,1),(924,'ANL00053','en','principal component analysis',1,0),(925,'ANL00054','en','NOE',1,1),(926,'ANL00054','en','NMR Nuclear Overhauser Effect',1,0),(927,'ANL00054','en','Nuclear Overhauser Effect',0,0),(928,'ANL00055','en','Water shell',1,0),(929,'ANL00056','en','Coarse-Grained Elastic Mesoscopic Model Flexibility',1,0),(930,'ANL00057','en','Coarse-Grained Worm-like Chain Model Flexibility',1,0),(931,'ANL00058','en','Ramachandran Plot',1,0),(932,'ANL00059','en','SAS',1,1),(933,'ANL00059','en','Solvent accessible surface',1,0),(934,'ANL00059','en','Accessible surface area',1,0),(935,'ANL00060','en','LCPO SAS',1,1),(936,'ANL00061','en','Shrake-Rupley SAS',1,0),(937,'ANL00062','en','TM-score',1,0),(938,'ANL00063','en','Molecular mechanics/Poisson-Boltzman Surface Area',1,0),(939,'ANL00063','en','MM-PBSA',1,1),(940,'ANL00064','en','Molecular mechanics/Generalized-Boltzman Surface Area',1,0),(941,'ANL00064','en','MM-GBSA',1,1),(942,'ANL00001','en','Analysis method',1,0),(943,'TSK01000','en','Analysis task',1,0),(944,'TSK01000','en','Analysis',0,0),(946,'HW01300','en','Hard disk',1,0),(947,'HW01300','en','Hard drive',1,0),(948,'HW01100','en','Central processing unit',0,0),(949,'HW01200','en','Graphics processing unit',0,0);
/*!40000 ALTER TABLE `CONCEPT_TERM` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `DESCRIPTION`
--

DROP TABLE IF EXISTS `DESCRIPTION`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `DESCRIPTION` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `CUI` varchar(50) DEFAULT NULL,
  `LANG` varchar(50) NOT NULL,
  `STR` text NOT NULL,
  PRIMARY KEY (`ID`),
  UNIQUE KEY `UQ_DESCRIPTION_ID` (`ID`),
  KEY `CUI` (`CUI`)
) ENGINE=MyISAM AUTO_INCREMENT=678 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `DESCRIPTION`
--

LOCK TABLES `DESCRIPTION` WRITE;
/*!40000 ALTER TABLE `DESCRIPTION` DISABLE KEYS */;
INSERT INTO `DESCRIPTION` VALUES (1,'C40017','en','Coupled-cluster singles, doubles, triples, and quadruples'),(2,'C40016','en','Coupled-cluster singles, doubles, and triples'),(3,'C40019','en','Completely renormalized CCSD(T) method'),(4,'C40018','en','CCSD and perturbative connected triples'),(5,'C40013','en','Coupled-cluster singles and doubles'),(6,'C40012','en','Coupled-cluster doubles'),(7,'C40015','en','Linearized coupled-cluster singles and doubles'),(8,'C40014','en','Linearized coupled-cluster doubles'),(9,'C40010','en','Configuration interaction singles, doubles, and triples'),(10,'QM01110','en','Correlation-consistent basis set'),(11,'C40011','en','Configuration interaction singles, doubles, triples, and quadruples'),(12,'C11101','en','Andersen thermostat'),(13,'C40008','en','Moeller-Plesset perturbation theory (fifth-order)'),(14,'C40007','en','Moeller-Plesset perturbation theory (fourth-order)'),(15,'C11103','en','Langevin thermostat'),(16,'C40006','en','Moeller-Plesset perturbation theory (third-order)'),(17,'C11102','en','Berendsen thermostat'),(18,'C40005','en','Moeller-Plesset perturbation theory (second-order)'),(19,'C40004','en','Self-consistent field'),(20,'C40003','en','Restricted open-shell Hartree-Fock'),(21,'C40002','en','Unrestricted Hartree-Fock'),(22,'C40001','en','Hartree-Fock'),(23,'MTH14002','en','Hybrid method partitioning the space into a QM domain and an MM domain.'),(24,'MTH14001','en','Computational quantum chemistry method combining multiple level of theories.'),(25,'MTH14000','en','Hybrid method'),(26,'C40009','en','Configuration interaction singles and doubles'),(27,'MTH10000','en','Computational method'),(28,'QM01120','en','Pople basis set'),(29,'C11104','en','V-rescale thermostat'),(30,'C11105','en','Nose thermostat'),(31,'C11106','en','Nose-Hoover thermostat'),(32,'C11107','en','Nose-Poincare thermostat'),(33,'QM01130','en','Polarisation-consistent basis set'),(34,'PRM00011','en','Quantum chemistry parameter set'),(35,'C17104','en','OPLS (Optimized Potential for Liquid Simulations) force field'),(36,'C17200','en','TIP3P 3-site water model: each atom gets assigned a point charge.'),(37,'QM01140','en','Effective core potential basis set'),(38,'C17102','en','OPLS (Optimized Potential for Liquid Simulations) UA (United Atom) force field'),(39,'PRM00301','en','Molecular dynamics setting'),(40,'C17103','en','OPLS (Optimized Potential for Liquid Simulations) force field'),(41,'C17207','en','SPC water model (Single Point Charge)'),(42,'C17208','en','Flexible SPC water model (Single Point Charge)'),(43,'C17205','en','TIP5P 5-site water model: negative charge placed on 2 dummy atoms.'),(44,'C17206','en','TIP6P 6-site water model: combines TIP4P and TIP5P models.'),(45,'C17203','en','TIP4P 4-site water model for simulation of solid water ice.'),(46,'C17101','en','OPLS (Optimized Potential for Liquid Simulations) AA (All-Atom) force field'),(47,'C17204','en','TIP4P 4-site water model for simulating the entire phase diagram of condensed water.'),(48,'PRM00201','en','Method setting'),(49,'C17201','en','TIP4P 4-site water model: a negative charge is placed on a dummy atom.'),(50,'C17202','en','TIP4P 4-site water model for use with Ewald summation methods.'),(51,'PRM00001','en','Parameter set'),(52,'C17209','en','SPC water model with average polarization correction to the potential energy function'),(53,'C17303','en','Allner ion force field parameter set (2012)'),(54,'PRM00101','en','Setting'),(55,'C17302','en','Joung/Cheatham ion force field parameter set (2008)'),(56,'C17301','en','Beglov/Roux ion force field parameter set'),(57,'C17300','en','Aqvist ion force field parameter set'),(58,'C40021','en','Quadratic configuration interaction singles & doubles and perturbative connected triples'),(59,'C40020','en','Quadratic configuration interaction singles & doubles'),(60,'C17304','en','Merz ion force field parameter set (2013)'),(61,'C10061','en','aug-cc-pV(Q+d)Z correlation-consistent polarized basis set'),(62,'C10060','en','aug-cc-pV(T+d)Z correlation-consistent polarized basis set'),(63,'C10065','en','TZ2PD Triple Zeta + polarization + Diffuse on all atoms'),(64,'C10064','en','DZP Double Zeta + polarization + Tight core'),(65,'C10063','en','aug-cc-pV(6+d)Z correlation-consistent polarized basis set'),(66,'C10062','en','aug-cc-pV(5+d)Z correlation-consistent polarized basis set'),(67,'C10069','en','aug-pc-1 basis set'),(68,'C10068','en','aug-pc-0 basis set'),(69,'C10067','en','5Z4PD Quintuple Zeta + polarization + Diffuse on all atoms'),(70,'QM01150','en','Ahlrichs basis set'),(71,'C10066','en','QZ3PD Quadruple Zeta + polarization + Diffuse on all atoms'),(72,'TSK00101','en','Quantum calculation'),(73,'HW01200','en','Graphics processing unit are generally used to manipulate computer graphics, although they can also be used for general-purpose computations.'),(74,'FS01110','en','File format'),(76,'C10070','en','aug-pc-2 basis set'),(77,'C10072','en','aug-pc-4 basis set'),(78,'C10071','en','aug-pc-3 basis set'),(79,'C10074','en','1P polarization functions'),(80,'HW01100','en','Central processing unit of a computer or other processing system (e.g. mobile phone, tablet).'),(81,'C10073','en','DFT Coulomb fitting'),(82,'C10076','en','VDZ Valence Double Zeta: 2 Funct.\'s/Valence AO'),(83,'C10075','en','VDZP Valence Double Zeta + Polarization on all atoms'),(84,'C10077','en','VTZ Valence Triple Zeta: 3 Funct.\'s/Valence AO'),(85,'SW01100','en','Operating system'),(86,'FF00014','en','OPEP coarse-grain force field'),(87,'FF00013','en','OPLS (Optimized Potential for Liquid Simulations) force field'),(88,'FF00016','en','Water model'),(89,'FF00015','en','Ion force field parameter set'),(90,'SW01200','en','Software package'),(91,'QM02000','en','Spin multiplicity'),(92,'C11002','en','Basis set where valence orbitals are represented by more than one basis function. They include Pople and correlation-consistent basis sets.'),(93,'C11001','en','Basis set using a fixed number of Gaussian primitive functions to represent core and valence orbitals.'),(94,'C11004','en','Real-space basis sets: functions are centered on a uniform mesh in real space.'),(95,'C11003','en','Basis set using a finite number of plane-wave functions.'),(96,'C10047','en','aug-cc-pV8Z correlation-consistent polarized basis set'),(97,'C10046','en','aug-cc-pV6Z correlation-consistent polarized basis set'),(98,'C10045','en','aug-cc-pV5Z correlation-consistent polarized basis set'),(99,'C10044','en','aug-cc-pVQZ correlation-consistent polarized basis set'),(100,'C10043','en','aug-cc-pVTZ correlation-consistent polarized basis set'),(101,'C10042','en','aug-cc-pVDZ correlation-consistent polarized basis set'),(102,'C10041','en','cc-pV(6+d)Z correlation-consistent polarized basis set'),(103,'C10040','en','cc-pV(5+d)Z correlation-consistent polarized basis set'),(104,'C10048','en','aug-cc-pV9Z correlation-consistent polarized basis set'),(105,'C10049','en','aug-cc-pVDZ-RI correlation-consistent polarized basis set'),(106,'QM01000','en','Basis set'),(107,'C10050','en','aug-cc-pVTZ-RI correlation-consistent polarized basis set'),(108,'C10056','en','aug-cc-pCVQZ correlation-consistent polarized basis set'),(109,'C10055','en','aug-cc-pCVTZ correlation-consistent polarized basis set'),(110,'C10058','en','aug-cc-pCV6Z correlation-consistent polarized basis set'),(111,'C10057','en','aug-cc-pCV5Z correlation-consistent polarized basis set'),(112,'C10052','en','aug-cc-pV5Z-RI correlation-consistent polarized basis set'),(113,'C10051','en','aug-cc-pVQZ-RI correlation-consistent polarized basis set'),(114,'C10054','en','aug-cc-pCVDZ correlation-consistent polarized basis set'),(115,'C10053','en','aug-cc-pV6Z-RI correlation-consistent polarized basis set'),(116,'C10059','en','aug-cc-pV(D+d)Z correlation-consistent polarized basis set'),(117,'FS01100','en','File'),(118,'MTH12200','en','Semi-empirical method restricted to Pi electrons'),(119,'C12021','en','Evaluate the derivative of the energy with respect to nuclear coordinates.'),(120,'QM04004','en','Kerker pseudopotential scheme'),(121,'C12020','en','Minimize the energy by varying the molecular structure.'),(122,'C12023','en','Compute gradient and second derivatives.'),(123,'QM04002','en','Hamann pseudopotential'),(124,'C12022','en','Evaluate heat of formation'),(125,'QM04003','en','Bachelet-Hamann-SchlÃ¼ter pseudopotential'),(126,'C12025','en','Hyperfine coupling constants (anisotropic)'),(127,'C12024','en','High accuracy energies'),(128,'C12027','en','Ionization potentials'),(129,'C12026','en','Hyperfine spectra tensors (including g tensors)'),(130,'C12028','en','Infrared spectra'),(131,'C12029','en','Reaction path following.'),(132,'QM04001','en','Troullier-Martins pseudopotential'),(133,'QM04000','en','Pseudo-potential'),(134,'MTH12000','en','Semi-empirical method'),(135,'C12030','en','Find the maximum energy along a specific reaction path.'),(136,'C12034','en','Molecular orbitals'),(137,'C12033','en','Locate minimum energy crossing point on the intersection seam of two potential energy surfaces'),(138,'C12032','en','Polarizabilities and hyperpolarizabilities.'),(139,'C12031','en','Pre-resonance Raman spectra'),(140,'C12038','en','Compute forces on the nuclei.'),(141,'C12037','en','NMR spin-spin coupling constants'),(142,'C12036','en','NMR shielding and chemical shifts'),(143,'C12035','en','Multipole moments'),(144,'C12039','en','Optical rotations'),(147,'SYS00101','en','Implicit solvent'),(148,'SYS00102','en','Explicit solvent'),(149,'SYS00103','en','In-vacuo'),(150,'C12041','en','Raman spectra'),(151,'C12040','en','Compute radiative transition moment'),(152,'C12047','en','Thermochemical analysis'),(153,'C12046','en','Compute spin-orbit coupling.'),(154,'C12049','en','Analytic computation of time dependent polarizabilities.'),(155,'MD00307','en','Beeman integrator'),(156,'MD00306','en','Leap-frog integrator'),(157,'C12043','en','Conduct a search for a transition state (or saddle point).'),(158,'C12042','en','Raman optical activity'),(159,'C12045','en','Evaluate the single point energy.'),(160,'C12044','en','Potential energy surface scan.'),(161,'C14002','en','Particle Mesh Ewald (PME)'),(162,'C14003','en','Particle-Particle Particle Mesh (P3M) is a Fourier-based Ewald summation method to calculate potentials in N-body simulations'),(163,'C14001','en','Plain cutoff used to truncate any Coulombic interactions after a finite distance'),(164,'C14004','en','The reaction field method for electrostatics assumes a uniform dielectric constant beyond a short-range cutoff value'),(165,'SW00400','en','CHARMM (Chemistry at HARvard Molecular Mechanics) MD software package'),(166,'C12052','en','Vibration-rotation coupling'),(167,'C12051','en','UV/Visible spectra'),(168,'C12050','en','Trace gradient extremal'),(169,'C12056','en','Calculate the properties for the wave function.'),(170,'C12055','en','Test wavefunction stability'),(171,'C12054','en','Compute molecular volume'),(172,'C12053','en','Vibrational circular dichroism'),(173,'QM05200','en','Pure functional'),(174,'MTH12100','en','Semi-empirical method restricted to all-valence electrons'),(175,'PRM00031','en','Minimization parameter set'),(176,'QM05100','en','Hybrid exchange-correlational functional'),(177,'QM05000','en','Exchange-correlational functional'),(676,'TSK10003','en','Set of sequential MD tasks used to equilibrate the system.'),(182,'PRM00021','en','Molecular dynamics parameter set'),(183,'HW01000','en','Physical, mechanical components of a computer or data processing system.'),(184,'MD00400','en','Pressure control'),(186,'MD00303','en','Adams-Bashforth-Moulton integrator'),(188,'MD00302','en','Runge-Kutta integrator'),(677,'TSK10004','en','Set of production MD tasks run after equilibration'),(190,'MD00305','en','Verlet velocity integrator'),(192,'QM05300','en','Exchange-only functional'),(193,'MD00304','en','Verlet integrator'),(194,'MD00301','en','Euler integrator'),(196,'MD00300','en','Molecular mechanics integrator'),(197,'C40363','en','DFT with SAOP exchange-correlation functional'),(198,'C12002','en','Antiferromagnetic coupling'),(200,'C40362','en','DFT with S12h exchange-correlation functional'),(201,'C12003','en','Atomic charges'),(202,'SW00300','en','AMBER (Assisted Model Building and Energy Refinement) MD software package'),(204,'C40361','en','DFT with S12g exchange-correlation functional'),(205,'C12004','en','Locate conical intersection point on the intersection seam of two potential energy surfaces.'),(674,'TSK10001','en','Set of sequential minimization tasks.'),(207,'C40360','en','DFT with RPBE exchange-correlation functional'),(208,'C12005','en','Delta G of solvation'),(209,'C12001','en','Anharmonic vibrational corrections'),(210,'C12101','en','Spectroscopic property'),(211,'C12102','en','Thermodynamics'),(212,'C12103','en','Electronic property'),(213,'C12104','en','Physical property'),(214,'C16001','en','Accelerated Molecular Dynamics'),(216,'C16002','en','Adaptive Force Bias'),(217,'SW00500','en','CP2K software package'),(219,'C16003','en','Canonical adiabatic free energy sampling (CAFES)'),(220,'C40368','en','DFT with TPSSh exchange-correlation functional'),(221,'C16004','en','Forward flux simulation'),(223,'C40369','en','DFT with VSXC pure functional'),(224,'C16005','en','Locally enhanced sampling'),(226,'C40366','en','DFT with tHCTH pure functional'),(227,'C16006','en','Markov chain model'),(228,'C12007','en','Dynamics trajectory calculation.'),(230,'C40367','en','DFT with tHCTHhyb exchange-correlation functional'),(231,'C16007','en','Max-flux approach'),(232,'C12006','en','Dipole moment'),(234,'C40364','en','DFT with SSB-D exchange-correlation functional'),(235,'C16008','en','Metadynamics'),(236,'C12009','en','Electron affinities'),(675,'TSK10002','en','Set of sequential MD tasks where the system is incrementally increased to the reference temperature'),(238,'C40365','en','DFT with tau-HCTH pure functional'),(239,'C16009','en','Milestoning'),(240,'C12008','en','Follow dynamic reaction coordinate.'),(241,'C40372','en','DFT with wB97XD exchange-correlation functional'),(242,'C12015','en','Perform energy decomposition analysis.'),(243,'C40371','en','DFT with wB97X exchange-correlation functional'),(244,'C12016','en','Fragment Molecular Orbital (FMO) geometry optimization'),(245,'C40374','en','DFT with XAlpha exchange functional'),(246,'C12013','en','Electrostatic potential-derived charges'),(247,'C40373','en','DFT with X3LYP exchange-correlation functional'),(248,'C12014','en','Electronic transition band shape'),(249,'SYS10000','en','Physical system representing a single molecule or multiple molecules at a subatomtic, atomic, or coarser-grain level.'),(250,'C12011','en','Electronic circular dichroism'),(251,'C12012','en','Electrostatic potential'),(252,'C40370','en','DFT with wB97 exchange-correlation functional'),(253,'C12010','en','Electron density'),(254,'C16011','en','Neural networks'),(255,'C16012','en','Nudged elastic band'),(256,'C16010','en','Multi-state Bennett Acceptance Ratio (MBAR)'),(673,'TSK10000','en','Set of sequential computational tasks.'),(258,'C16015','en','Steered dynamics'),(260,'C16016','en','String method'),(262,'C16013','en','Reference potential spatial warping algorithm'),(264,'C16014','en','Replica-Exchange molecular dynamics'),(266,'C16019','en','Transition path sampling'),(268,'C12019','en','Evaluate up to 3rd nuclear derivatives, by finite differencing of Hessians.'),(269,'C12018','en','Frequency and thermochemical analysis.'),(270,'C16017','en','Swarm-of-trajectories'),(271,'C12017','en','Performs the free state Fragment Molecular Orbital (FMO) calculation'),(272,'C16018','en','Targeted molecular dynamics'),(273,'C40339','en','DFT with M06HF exchange-correlation functional'),(274,'SW00119','en','Firefly (previously known as PC GAMESS) QM software package'),(275,'C40334','en','DFT with LC-PBE exchange-correlation functional'),(276,'PRM00650','en','Dielectrically-consistent RISM'),(277,'SW00118','en','NWChem QM software package'),(278,'C40333','en','DFT with LC-wPBE exchange-correlation functional'),(279,'SW00117','en','NAMD (Not Another Molecular Dynamics program) software package'),(280,'C40332','en','DFT with LB94 exchange-correlation functional'),(281,'SW00116','en','MOPAC (Molecular Orbital PACkage) semiempirical quantum chemistry software package'),(282,'C40331','en','DFT with KT1 exchange-correlation functional'),(283,'C40338','en','DFT with M06 exchange-correlation functional'),(284,'C40337','en','DFT with M05-2X exchange-correlation functional'),(285,'C40336','en','DFT with M05 exchange-correlation functional'),(286,'C40335','en','DFT with LDA exchange-correlation functional'),(287,'SW00110','en','Gaussian QM software package. Marketed by Gaussian, Inc.'),(288,'SW00111','en','GROMACS MD software package'),(289,'SW00114','en','Jaguar QM software package. Marketed by Schrodinger, Inc.'),(290,'SW00115','en','LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) MD software package'),(291,'C40330','en','DFT with HSEhPBE exchange-correlation functional'),(292,'SW00112','en','GROMOS (GROningen MOlecular Simulation package) MD software package'),(293,'C40329','en','DFT with HSEh1PBE exchange-correlation functional'),(294,'PRM00649','en','Extended RISM'),(295,'C40328','en','DFT with HSE2PBE exchange-correlation functional'),(296,'PRM00647','en','Analytical Linearized Poisson-Boltzmann'),(297,'PRM00648','en','Reference Interaction Site Model of molecular solvation'),(298,'PRM00646','en','GB model described by Mongan, Simmerling, McCammon, Case and Onufriev. Pairwise correction term to GB HCT to approximate a molecular surface dielectric boundary; that is to eliminate interstitial regions of high dielectric smaller than a solvent molecule.'),(299,'C40321','en','DFT with HCTH pure functional'),(300,'C40320','en','DFT with DSD-PBEBP86 exchange-correlation functional'),(301,'C40323','en','DFT with HCTH147 pure functional'),(302,'PRM00501','en','Minimization setting'),(303,'C40322','en','DFT with HCTH93 pure functional'),(304,'C40325','en','DFT with HFB exchange functional'),(305,'C40324','en','DFT with HCTH407 pure functional'),(306,'C40327','en','DFT with HSE exchange-correlation functional'),(307,'C40326','en','DFT with HFS exchange functional'),(308,'SW00100','en','Software package for biomolecular simulations and protein folding'),(309,'SW00600','en','CPMD (Car-Parinello Molecular Dynamics) software package'),(310,'SW00200','en','ADF (Amsterdam Density Functional) software package'),(311,'PRM00632','en','Accessible surface area-based method'),(312,'PRM00633','en','Poisson-Boltzmann solvent'),(313,'PRM00634','en','Generalized Born model'),(314,'PRM00635','en','Ad hoc fast solvation model'),(315,'SYS00201','en','Periodic boundary conditions'),(316,'SYS00202','en','Non-periodic boundary conditions'),(317,'C40359','en','DFT with RPA exchange-correlation functional'),(318,'C40358','en','DFT with revTPSS-D exchange-correlation functional'),(319,'C40357','en','DFT with revTPSS exchange-correlation functional'),(320,'C40356','en','DFT with revPBE exchange-correlation functional'),(321,'C40355','en','DFT with PWPB95-D3 exchange-correlation functional'),(322,'MD00500','en','Temperature control'),(323,'C40354','en','DFT with PW6B95 exchange-correlation functional'),(324,'SYS00200','en','Boundary conditions'),(325,'C40353','en','DFT with PW91 exchange-correlation functional'),(326,'PRM00631','en','Hybrid implicit/explicit solvent'),(327,'C40351','en','DFT with PBE1PBE exchange-correlation functional'),(328,'C40352','en','DFT with PBEh1PBE exchange-correlation functional'),(329,'C40350','en','DFT with PBE exchange-correlation functional'),(330,'MD00700','en','Collision frequency'),(331,'PRM00621','en','Solvent model'),(332,'C16022','en','Umbrella sampling'),(333,'C16021','en','Transition interface sampling'),(334,'C16020','en','Transition state ensembles'),(335,'C40347','en','DFT with O3LYP exchange-correlation functional'),(336,'C40346','en','DFT with mPW3PBE exchange-correlation functional'),(337,'C40349','en','DFT with optB88-vdW exchange-correlation functional'),(338,'C40348','en','DFT with OLYP exchange-correlation functional'),(339,'C40343','en','DFT with mPW1LYP exchange-correlation functional'),(340,'C40342','en','DFT with mPW1k exchange-correlation functional'),(341,'C40345','en','DFT with mPW1PW91 exchange-correlation functional'),(342,'C40344','en','DFT with mPW1PBE exchange-correlation functional'),(343,'C40340','en','DFT with M06L pure functional'),(344,'C40341','en','DFT with M062X exchange-correlation functional'),(345,'C30019','en','Comma-separated value (CSV) file format'),(346,'MTH11300','en','Enhanced sampling method'),(347,'C30017','en','CHARMM output file format'),(348,'C30018','en','Chemical Markup Language (CML) file format'),(349,'C30015','en','CHARMM coordinates file format'),(350,'C30016','en','CHARMM input file format'),(351,'C30013','en','AMBER trajectory (NetCDF) format'),(352,'C30014','en','Bitmap image file'),(353,'C30012','en','AMBER trajectory (Scripps Binpos) format. Binpos files have a 4 byte header followed by binpos frame records. Each frame record is an integer (number of atoms) followed by 3*natom.'),(354,'C30011','en','AMBER trajectory (binary) format'),(355,'C17504','en','Coarse-grain force field'),(356,'C30010','en','AMBER trajectory (ASCII) format'),(357,'C17503','en','Reactive force field'),(358,'C17502','en','Polarizable force field'),(359,'C17501','en','Classical force field'),(360,'MTH11200','en','Minimization'),(361,'MD00604','en','Generalized ensembles: use of multiple ensembles, typically run in parallel, to overcome multipl-minima problems.'),(362,'C30006','en','AMBER MD output format'),(363,'MD00603','en','Isothermal-isobaric ensemble: the ensemble has a fixed number of particles N, a fixed pressure P, and a fixed temperature T (NPT).'),(364,'C30007','en','AMBER parameter/topology format'),(365,'C30008','en','AMBER restart format'),(366,'C30002','en','ADF output file format'),(367,'C30003','en','ADF Tape41 file format'),(368,'PRM00601','en','Molecular system setting'),(369,'C30004','en','AMBER OFF library format'),(370,'PRM00600','en','System parameter.'),(371,'C30005','en','AMBER MD input format'),(372,'C30001','en','ADF input file format'),(374,'C40315','en','DFT with BLYP exchange-correlation functional'),(375,'C40316','en','DFT with BMK exchange-correlation functional'),(376,'C40313','en','DFT with BHandH exchange-correlation functional'),(377,'C40314','en','DFT with BHandHLYP exchange-correlation functional'),(378,'C40311','en','DFT with B971 exchange-correlation functional'),(379,'C40312','en','DFT with B972 exchange-correlation functional'),(380,'MD00100','en','Restraint'),(381,'C40310','en','DFT with B97-D pure functional'),(382,'C30037','en','GROMACS restart'),(383,'C30038','en','GROMACS system topology'),(384,'MD00200','en','Constraint'),(385,'C30035','en','GROMACS MD input'),(386,'C30036','en','GROMACS MD output'),(387,'C40319','en','DFT with DCD-BLYP exchange-correlation functional'),(388,'C40317','en','DFT with BP86 exchange-correlation functional'),(389,'C30039','en','GROMACS trajectory'),(390,'C40318','en','DFT with CAM-B3LYP exchange-correlation functional'),(391,'MD00601','en','Microcanonical ensemble (NVE): the ensemble has a fixed number of particles N, a fixed volume V, and a fixed energy E.'),(392,'C30030','en','GAUSSIAN restart file format'),(393,'MD00602','en','Canonical ensemble (NVT): the ensemble has a fixed number of particles N, a fixed volume V, and a fixed temperature T.'),(394,'MD00600','en','Ensemble'),(395,'C30034','en','GROMACS index file format'),(396,'C30033','en','GROMACS include topology'),(397,'C30032','en','GROMACS gro coordinate file format'),(398,'C30031','en','GIF (Graphics Interchange File) image file'),(399,'C40302','en','DFT with B1B95 exchange-correlation functional'),(400,'C40303','en','DFT with B2PLYP exchange-correlation functional'),(401,'C40304','en','DFT with B3LYP exchange-correlation functional'),(402,'C40305','en','DFT with B3LYP* exchange-correlation functional'),(403,'SW00127','en','VASP (Vienna Ab-initio Simulation Package) software package'),(404,'SW00128','en','X-PLOR software package'),(405,'C40300','en','DFT with APBE exchange-correlation functional'),(406,'C40301','en','DFT with B1LYP exchange-correlation functional'),(407,'C30024','en','GAMESS-UK input file format'),(408,'C30025','en','GAMESS-UK output file format'),(409,'C30026','en','GAUSSIAN chk file format'),(410,'C30027','en','GAUSSIAN input file format (.com)'),(411,'C40306','en','DFT with B3LYP-D exchange-correlation functional'),(412,'C30028','en','GAUSSIAN cube file format'),(413,'C40307','en','DFT with B3P86 exchange-correlation functional'),(414,'C30029','en','GAUSSIAN log/output file format'),(415,'C40308','en','DFT with B3PW91 exchange-correlation functional'),(416,'C40309','en','DFT with B98 exchange-correlation functional'),(417,'C30021','en','Fasta sequence file format'),(418,'C30020','en','DCD trajectory file format'),(419,'C30023','en','GAMESS output file format'),(420,'C30022','en','GAMESS input file format'),(421,'SW00124','en','TINKER software package'),(422,'SW00123','en','Software package, marketed by Wavefunction, Inc.'),(423,'SW00126','en','TURBOMOLE software package'),(424,'SW00125','en','Tremolo-X software package'),(425,'SW00120','en','Q-Chem QM software package'),(426,'SW00122','en','SIESTA (Spanish Initiative for Electronic Simulations with Thousand of Atoms) software package'),(427,'SW00121','en','Quantum ESPRESSO (opEn-Source Package for Research in Electronic Structure, Simulation, and Optimization) QM software package'),(672,'SYS11000','en','Box defining the limits of the 3D region where the molecular system is being represented.'),(429,'CPE00002','en','Computing environment'),(430,'FS01000','en','File system'),(431,'C30053','en','NWChem topology file format (.top)'),(432,'C30054','en','Protein Data Bank (PDB) file format'),(433,'C13001','en','LINCS (Linear Constraint Solver)'),(434,'C30055','en','PNG (Portable Networks Image) image file format'),(435,'C13002','en','M-SHAKE is an extension of SHAKE'),(436,'C30056','en','Protein Structure File (PSF) format'),(437,'C13003','en','P-SHAKE is an extension of SHAKE'),(438,'C13004','en','SETTLE'),(439,'C10039','en','cc-pV(Q+d)Z correlation-consistent polarized basis set'),(440,'C30050','en','NWChem input file format (.nw)'),(441,'C13005','en','SHAKE'),(442,'C10038','en','cc-pV(T+d)Z correlation-consistent polarized basis set'),(443,'C30051','en','NWChem output file format'),(445,'C13006','en','SHAPE is a multicenter analog of SHAKE for constraining rigid bodies of three or more centers.'),(446,'C10037','en','cc-pV(D+d)Z correlation-consistent polarized basis set'),(447,'C30052','en','NWChem restart file format (.rst)'),(448,'C13008','en','WIGGLE is an extension of SHAKE'),(449,'C10035','en','cc-pCV5Z correlation-consistent polarized basis set'),(450,'C13007','en','RATTLE is an extension of SHAKE'),(451,'C10036','en','cc-pCV6Z correlation-consistent polarized basis set'),(452,'C10033','en','cc-pCVTZ correlation-consistent polarized basis set'),(453,'C10034','en','cc-pCVQZ correlation-consistent polarized basis set'),(454,'C10031','en','cc-pV6Z-RI correlation-consistent polarized basis set'),(455,'C30058','en','Q-Chem input file format'),(456,'C10032','en','cc-pCVDZ correlation-consistent polarized basis set'),(457,'C30057','en','Ptraj script'),(458,'C10030','en','cc-pV5Z-RI correlation-consistent polarized basis set'),(459,'C30059','en','Q-Chem output file format'),(460,'QM01100','en','Atomic basis set'),(461,'TSK00201','en','Molecular dynamics run'),(462,'C30044','en','JPEG (Joint Photographic Experts Group) image file'),(463,'MTH13000','en','Ab initio method'),(464,'C30045','en','Mol2 file format'),(465,'C30042','en','Jaguar input file format'),(466,'C30043','en','Jaguar output file format'),(467,'C10027','en','cc-pVDZ-RI correlation-consistent polarized basis set'),(468,'C30040','en','HTML (HyperText Markup Language) file'),(469,'C10026','en','cc-pV9Z correlation-consistent polarized basis set'),(470,'C30041','en','HyperChem HIN file format'),(471,'C10029','en','cc-pVQZ-RI correlation-consistent polarized basis set'),(472,'C10028','en','cc-pVTZ-RI correlation-consistent polarized basis set'),(473,'C10022','en','cc-pVQZ correlation-consistent polarized basis set'),(474,'SW00800','en','GAMESS (General Atomic and Molecular Electronic Structure System) QM software package (UK variant)'),(475,'C10023','en','cc-pV5Z correlation-consistent polarized basis set'),(476,'C10024','en','cc-pV6Z correlation-consistent polarized basis set'),(477,'C10025','en','cc-pV8Z correlation-consistent polarized basis set'),(478,'C30049','en','NAMD configuration'),(479,'C30048','en','MOPAC Output file format'),(480,'C10020','en','cc-pVDZ correlation-consistent polarized basis set'),(481,'C30047','en','MOPAC Internal file format'),(482,'C10021','en','cc-pVTZ correlation-consistent polarized basis set'),(483,'C30046','en','MOPAC Cartesian file format'),(484,'MTH11220','en','Conjugate gradient'),(485,'C10018','en','LANL2TZ'),(486,'C10017','en','1P Polarization Functions'),(487,'C10016','en','DZP Double Zeta + Polarization + Diffuse ECP'),(488,'C10015','en','Lanl2DZ+2s2p2d2f'),(489,'MTH13010','en','Ab-initio molecular dynamics'),(490,'C10019','en','TZ triple zeta basis set designed for an ECP + f polarization'),(491,'C10010','en','Uncontracted basis set'),(492,'C10013','en','DZ Double Zeta Basis Set designed for an ECP'),(493,'C10014','en','LANL2DZ+1d1f '),(494,'C10011','en','Uncontracted basis set + f polarization'),(495,'C10012','en','Lanl2-[6s4p4d2f]'),(496,'C11011','en','Andersen barostat'),(497,'MTH11210','en','Steepest descent'),(498,'C11012','en','Berendsen barostat'),(499,'C11013','en','Hoover barostat'),(500,'C17250','en','Reactive force field'),(501,'C11014','en','Parinello-Rahman barostat'),(502,'MTH13026','en','Gaussian-4 method'),(503,'C10005','en','4-21G (Pople basis set)'),(504,'C30062','en','Tagged Image File Format (TIFF)'),(505,'C17350','en','Optimized Potential for Efficient Structure Prediction (OPEP) version 1'),(506,'MTH13025','en','Gaussian-3 method'),(507,'C10004','en','3-21G (Pople basis set)'),(508,'C30063','en','TINKER XYZ file format'),(509,'MTH13024','en','Gaussian-2 method'),(510,'C10007','en','6-21G (Pople basis set)'),(511,'C30060','en','MDL Structure Data Format (SDF)'),(512,'MTH13023','en','Gaussian-1 method'),(513,'C10006','en','4-31G (Pople basis set)'),(514,'C30061','en','Text file (ASCII)'),(515,'QM03000','en','Frozen core'),(516,'C17354','en','Residue-based coarse-grain forcefield for biomolecular simulations'),(517,'MTH13022','en','Gaussian-n method'),(518,'C10009','en','6-311G (Pople basis set)'),(519,'TSK00001','en','Computational task'),(520,'MTH13021','en','Computational chemistry method that aims for high accuracy by combining the results of several calculations. These methods combine methods with a high level of theory and a small basis set with methods that employ lower levels of theory with larger basis sets.'),(521,'C10008','en','6-31G (Pople basis set)'),(522,'C17353','en','Optimized Potential for Efficient Structure Prediction (OPEP) version 4'),(523,'C30064','en','TURBOMOLE Coordinate file format'),(524,'C17352','en','Optimized Potential for Efficient Structure Prediction (OPEP) version 3'),(525,'MTH13020','en','Post-Hartree-Fock method'),(526,'C30065','en','XML (eXtensible Markup Language) file'),(527,'C17351','en','Optimized Potential for Efficient Structure Prediction (OPEP) version 2'),(528,'C17151','en','GROMOS 43a1 force field'),(529,'C17152','en','GROMOS 43b1 (vacuum) force field'),(530,'MTH13029','en','Composite method similar to G2 and G3 but contains an MP2 extrapolation to the complete basis set limit as one step.'),(531,'C10000','en','STO-2G minimal basis set'),(532,'C17155','en','GROMOS 45A4 force field'),(533,'C10001','en','STO-3G minimal basis set'),(534,'C17156','en','GROMOS 53A5 force field'),(535,'MTH13027','en','Computational approach developed for calculating accurate heats of formation of uncharged, closed-shell molecules comprising H, C, N, O, F, Si, P, S, Cl and Br, within experimental error.'),(536,'C10002','en','STO-4G minimal basis set'),(537,'C17153','en','GROMOS 43a2 force field'),(538,'MTH13028','en','Method that utilizes the correlation consistent basis sets developed by Dunning and co-workers. It does not contain any empirically fitted term.'),(539,'C10003','en','STO-6G minimal basis set'),(540,'C17154','en','GROMOS 45A3 force field'),(541,'C17159','en','GROMOS 54B7 (vacuum) force field'),(542,'C17054','en','CHARMM General Force Field (CGenFF) for drug-like molecules'),(543,'C17157','en','GROMOS 53A6 force field'),(544,'C17158','en','GROMOS 54A7 force field'),(545,'C17052','en','CHARMM 27 force field'),(546,'C17053','en','CHARMM 36 force field'),(547,'C17050','en','CHARMM 19 force field'),(548,'C17051','en','CHARMM 22 force field'),(549,'MTH00010','en','Number of iterations'),(550,'MTH13030','en','The Weizmann-n ab initio methods (Wn, n=1-4) are highly-accurate composite theories devoid of empirical parameters.'),(551,'MTH00011','en','Number of time steps'),(552,'MTH13031','en','Weizmann-1 method'),(553,'MTH13032','en','Weizmann-2 method'),(554,'MTH13033','en','Weizmann-3 method'),(555,'MTH13034','en','Weizmann-4 method'),(556,'MTH13035','en','Perturbative treatment applicable to multireference CASCI-type wavefunctions. It can be considered as a generalization of the well-known second-order MÃ¸ller-Plesset perturbation theory to multireference Complete Active Space cases.'),(557,'MTH13036','en','Method based on the Complete Active Space Perturbation Theory (CASPT).'),(558,'MTH11000','en','Empirical method'),(559,'C17002','en','AMBER FF96 force field'),(560,'PRM00401','en','Quantum chemistry setting'),(561,'C17001','en','AMBER FF94 force field'),(562,'C17004','en','AMBER FF99SB force field (\'Stony Brook\' modifications to FF99 backbone torsions)'),(563,'C17003','en','AMBER FF99 force field'),(564,'C17006','en','AMBER FF99 phi force field (\'Sorin/Pande\' modifications to FF99 backbone torsions)'),(565,'C17005','en','AMBER FF99SB force field with modifications on the Chi1 torsion potentials'),(566,'C17008','en','AMBER FF02 additive (non-polarizable) force field'),(567,'C17007','en','AMBER FF99bsc0 force field (\'Barcelona\' changes to FF99 for nucleic acids)'),(568,'MD00900','en','Force field'),(569,'MTH11100','en','Classical molecular dynamics'),(570,'C17009','en','AMBER FF03 force field'),(571,'SW00700','en','Desmond MD software package developed by D.E. Shaw Research and marketed by Schrodinger, Inc.'),(572,'SW00900','en','GAMESS (General Atomic and Molecular Electronic Structure System) QM software package (US variant)'),(573,'MTH00100','en','Observed or computed property of a molecular system or subsystem.'),(574,'TSK00301','en','Minimization task'),(575,'C40200','en','Pariser-Parr-Pople'),(576,'C40201','en','CNDO/2'),(577,'C40202','en','INDO semi-empirical method'),(578,'C40203','en','NDDO semi-empirical method'),(579,'C40204','en','MINDO semi-empirical method'),(580,'C40205','en','MNDO semi-empirical method'),(581,'C40206','en','AM1 semi-empirical method'),(582,'MTH11120','en','Coarse-grain molecular dynamics'),(583,'C40207','en','PM3 semi-empirical method'),(584,'C40104','en','Multi-reference Coupled-cluster method'),(585,'C40208','en','PM6 semi-empirical method'),(586,'C40209','en','RM1 semi-empirical method'),(587,'C40100','en','Multi-configuration self-consistent field (MC-SCF)'),(588,'C40101','en','Complete active space SCF'),(589,'C40102','en','Restricted active space SCF'),(590,'C40103','en','Multi-reference Configuration Interaction'),(591,'MTH00020','en','Time step length'),(592,'MD00800','en','Model used to represent electrostatics (Coulombic) interactions.'),(593,'C40212','en','ZINDO semi-empirical method'),(594,'C17013','en','AMBER force field for carbohydrates and lipids'),(595,'C40213','en','SINDO semi-empirical method'),(596,'C17012','en','General Amber Force Field (GAFF) for small molecules'),(597,'C40210','en','SAM1 semi-empirical method'),(598,'C17011','en','AMBER FF12SB force field'),(599,'C40211','en','Sparkle/AM1 semi-empirical method'),(600,'C17010','en','AMBER FF10 force field'),(601,'C40214','en','PM3MM semi-empirical method (variation of PM3)'),(602,'C40215','en','PDDG semi-empirical method (variation of PM6)'),(603,'MTH11110','en','All-atom molecular dynamics'),(604,'C17015','en','AMBER AMOEBA (Atomic Mutlipole Optimized Energetics for Biomolecular Applications) polarizable force field'),(605,'C17014','en','AMBER force field for carbohydrates and lipids using lone pairs (extra points)'),(606,'ANL00002','en','Calculate two-dimensional RMSD (i.e. RMSD of each frame to every other frame).'),(607,'ANL00003','en','Diagonalize a previously calculated symmetric matrix to obtain eigenmodes.'),(608,'ANL00004','en','Analyze previously calculated eigenmodes.'),(609,'ANL00005','en','Calculate auto correlation functions for previously calculated vectors.'),(610,'ANL00006','en','Calculate time cross functions for previously calculated vectors.'),(611,'ANL00007','en','Compute the angle in degrees between the center of mass of the three atom specifications listed.'),(612,'ANL00008','en','Compute the atomic positional fluctuations (RMSF) or B-factors'),(613,'ANL00009','en','Attempt to map atoms between two reference structures based on overall structure and bonding.'),(614,'ANL00010','en','Compute the straight coordinate average of the coordinate sets read in.'),(615,'ANL00011','en','Look for atoms that are closer (and optionally further separated than) a specified distance.'),(616,'ANL00012','en','Check for close atomic overlaps and/or unusual bond lengths in structures.'),(617,'ANL00013','en','Group configurations from the MD trajectory into distinct sets.'),(618,'ANL00014','en','Group configurations from the MD trajectory by binning dihedral angles'),(619,'ANL00015','en','Find the number of atom interactions (and native contacts) within a given distance.'),(620,'ANL00016','en','Count the number of native contacts that were lost'),(621,'ANL00017','en','Calculate correlation coefficient between data sets.'),(622,'ANL00018','en','Calculate the mean-squared displacements vs. frame number.'),(623,'ANL00019','en','Calculate the dihedral values from the center of mass of the four specified atom masks.'),(624,'ANL00020','en','Calculate the distance in between the center of mass of the two atom specifications listed.'),(625,'ANL00021','en','Calculate distance RMSD (also known as DME).'),(626,'ANL00022','en','Create a density grid that has a count of the number of selected atoms in each grid cell.'),(627,'ANL00023','en','Keeps track of hydrogen bond distances'),(628,'ANL00024','en','Analyze hydrogen bond interaction energies'),(629,'ANL00025','en','Analyze base pair stacking energies'),(630,'ANL00026','en','Calculate j-coupling values from dihedral angles.'),(631,'ANL00027','en','Calculate the Connolly surface area.'),(632,'ANL00028','en','Calculate various nucleic-acid structure parameters.'),(633,'ANL00029','en','Calculate shear parameter for nucleic acid structures'),(634,'ANL00030','en','Calculate stretch parameter for nucleic acid structures'),(635,'ANL00031','en','Calculate stagger parameter for nucleic acid structures'),(636,'ANL00032','en','Calculate buckle parameter for nucleic acid structures'),(637,'ANL00033','en','Calculate propel parameter for nucleic acid structures'),(638,'ANL00034','en','Calculate opening parameter for nucleic acid structures'),(639,'ANL00035','en','Calculate shift parameter for nucleic acid structures'),(640,'ANL00036','en','Calculate slide parameter for nucleic acid structures'),(641,'ANL00037','en','Calculate rise parameter for nucleic acid structures'),(642,'ANL00038','en','Calculate tilt parameter for nucleic acid structures'),(643,'ANL00039','en','Calculate roll parameter for nucleic acid structures'),(644,'ANL00040','en','Calculate twist parameter for nucleic acid structures'),(645,'ANL00041','en','Calculate x-displacement parameter for nucleic acid structures'),(646,'ANL00042','en','Calculate y-displacement parameter for nucleic acid structures'),(647,'ANL00043','en','Align to the principal coordinate axis.'),(648,'ANL00044','en','Project coordinates along previously calculated eigenmodes.'),(649,'ANL00045','en','Compute the pucker for N-membered rings.'),(650,'ANL00046','en','Determine the radius of gyration for each frame.'),(651,'ANL00047','en','Compute a radial distribution function.'),(652,'ANL00048','en','Compute the RMSd to the first frame or a reference frame.'),(653,'ANL00049','en','Calculate RMSD correlation from the average RMSD of running-averaged coordinates over increasing window sizes.'),(654,'ANL00050','en','Calculate the running average of coordinates over a given window size.'),(655,'ANL00051','en','Determine the secondary structure of proteins.'),(656,'ANL00052','en','Count the number of secondary structure changes occurring in the trajectory.'),(657,'ANL00053','en','Principal component analysis'),(658,'ANL00054','en','Calculate NMR Nuclear Overhauser Effect'),(659,'ANL00055','en','Determine how many solvent models are within a given radius of the solute.'),(660,'ANL00056','en','Coarse-Grained elastic mesoscopic model flexibility analysis'),(661,'ANL00057','en','Coarse-Grained worm-like chain model flexibility analysis'),(662,'ANL00058','en','Generate Ramachandran plots to identifying mobile residues in the secondary structure'),(663,'ANL00059','en','Solvent-accessible surface calculation'),(664,'ANL00060','en','Calculate the surface area from linear combinations of pairwise overlaps.'),(665,'ANL00061','en','Solvent-accessible surface calculation using Shrake-Rupley algorithm'),(666,'ANL00062','en','Calculate the structural similarity of two protein models using TM-score algorithm'),(667,'ANL00063','en','Predict binding free energies using the Molecular Mechanics/Poisson-Boltzmann Surface Area (MM/PBSA) method'),(668,'ANL00064','en','Predict binding free energies using the Molecular Mechanics/Generalized Born Surface Area (MM/GBSA) method'),(669,'ANL00001','en','Analysis method'),(670,'TSK01000','en','Post-processing of simulated structures or QM calculations to summarize or identify certain properties of the original data.'),(671,'HW01300','en','A rigid magnetic disk mounted permanently in a drive unit that stores computer data.');
/*!40000 ALTER TABLE `DESCRIPTION` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `EXTERNAL_ONTOLOGY`
--

DROP TABLE IF EXISTS `EXTERNAL_ONTOLOGY`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `EXTERNAL_ONTOLOGY` (
  `ID` varchar(50) NOT NULL,
  `NAME` varchar(50) NOT NULL,
  `PREFIX` varchar(255) NOT NULL,
  `DOCUMENT_URL` varchar(255) DEFAULT NULL,
  `DESCRIPTION` text,
  PRIMARY KEY (`ID`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `EXTERNAL_ONTOLOGY`
--

LOCK TABLES `EXTERNAL_ONTOLOGY` WRITE;
/*!40000 ALTER TABLE `EXTERNAL_ONTOLOGY` DISABLE KEYS */;
INSERT INTO `EXTERNAL_ONTOLOGY` VALUES ('CHEBI','CHEBI','http://purl.obolibrary.org/obo/','ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/chebi.owl','Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on small chemical compounds.'),('IAO','IAO','http://purl.obolibrary.org/obo/','http://purl.obolibrary.org/obo/iao/iao-main.owl','Information Artifact Ontology'),('OBI','OBI','http://purl.obolibrary.org/obo/','http://purl.obolibrary.org/obo/obi.owl','Ontology for Biomedical Investigations'),('BFO','BFO','http://purl.obolibrary.org/obo/','http://purl.obolibrary.org/obo/bfo.owl','Basic Formal Ontology (BFO)'),('RO','RO','http://purl.obolibrary.org/obo/','http://purl.obolibrary.org/obo/ro.owl','OBO Relation Ontology (RO)'),('SO','SO','http://purl.obolibrary.org/obo/','http://purl.obolibrary.org/obo/so.owl','Sequence Ontology (SO)'),('CHEMINF','CHEMINF','http://semanticscience.org/resource/','http://semanticscience.org/ontology/cheminf.owl','Chemical Information Ontology (CHEMINF)');
/*!40000 ALTER TABLE `EXTERNAL_ONTOLOGY` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `RELATIONSHIP`
--

DROP TABLE IF EXISTS `RELATIONSHIP`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `RELATIONSHIP` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `CUI1` varchar(50) NOT NULL,
  `CUI2` varchar(50) NOT NULL,
  `REL_ID` varchar(50) NOT NULL,
  PRIMARY KEY (`ID`),
  UNIQUE KEY `UQ_RELATIONSHIP_ID` (`ID`),
  KEY `REL_ID` (`REL_ID`),
  KEY `CUI1` (`CUI1`),
  KEY `CUI2` (`CUI2`)
) ENGINE=MyISAM AUTO_INCREMENT=743 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `RELATIONSHIP`
--

LOCK TABLES `RELATIONSHIP` WRITE;
/*!40000 ALTER TABLE `RELATIONSHIP` DISABLE KEYS */;
INSERT INTO `RELATIONSHIP` VALUES (1,'C40017','MTH13004','IS_A'),(2,'C40016','MTH13004','IS_A'),(4,'C40019','MTH13004','IS_A'),(5,'C40018','MTH13004','IS_A'),(6,'C40013','MTH13004','IS_A'),(7,'C40012','MTH13004','IS_A'),(8,'C40015','MTH13004','IS_A'),(9,'C40014','MTH13004','IS_A'),(10,'C40010','MTH13003','IS_A'),(11,'QM01110','C11002','IS_A'),(12,'C40011','MTH13003','IS_A'),(13,'C11101','MD00500','IS_A'),(14,'C40008','MTH13002','IS_A'),(15,'C40007','MTH13002','IS_A'),(16,'C11103','MD00500','IS_A'),(17,'C40006','MTH13002','IS_A'),(18,'C11102','MD00500','IS_A'),(19,'C40005','MTH13002','IS_A'),(20,'C40004','MTH13001','IS_A'),(21,'C40003','MTH13001','IS_A'),(22,'C40002','MTH13001','IS_A'),(23,'C40001','MTH13001','IS_A'),(24,'MTH14002','MTH14000','IS_A'),(25,'MTH14001','MTH14000','IS_A'),(26,'MTH14000','MTH10000','IS_A'),(27,'C40009','MTH13003','IS_A'),(28,'QM01120','C11002','IS_A'),(29,'C11104','MD00500','IS_A'),(30,'C11105','MD00500','IS_A'),(31,'C11106','MD00500','IS_A'),(32,'C11107','MD00500','IS_A'),(33,'QM01130','C11002','IS_A'),(34,'PRM00011','PRM00001','IS_A'),(35,'C17104','FF00013','IS_A'),(36,'C17200','FF00016','IS_A'),(37,'QM01140','QM01000','IS_A'),(38,'C17102','FF00013','IS_A'),(39,'PRM00301','PRM00201','IS_A'),(40,'C17103','FF00013','IS_A'),(41,'C17207','FF00016','IS_A'),(42,'C17208','FF00016','IS_A'),(43,'C17205','FF00016','IS_A'),(44,'C17206','FF00016','IS_A'),(45,'C17203','FF00016','IS_A'),(46,'C17101','FF00013','IS_A'),(47,'C17204','FF00016','IS_A'),(48,'PRM00201','PRM00101','IS_A'),(49,'C17201','FF00016','IS_A'),(50,'C17202','FF00016','IS_A'),(51,'C17209','FF00016','IS_A'),(52,'C17303','FF00015','IS_A'),(53,'C17302','FF00015','IS_A'),(54,'C17301','FF00015','IS_A'),(55,'C17300','FF00015','IS_A'),(56,'C40021','C40020','IS_A'),(57,'C40020','MTH13020','IS_A'),(58,'C17304','FF00015','IS_A'),(59,'C10061','QM01110','IS_A'),(60,'C10060','QM01110','IS_A'),(61,'C10065','QM01110','IS_A'),(62,'C10064','QM01110','IS_A'),(63,'C10063','QM01110','IS_A'),(64,'C10062','QM01110','IS_A'),(65,'C10069','QM01130','IS_A'),(66,'C10068','QM01130','IS_A'),(67,'C10067','QM01110','IS_A'),(68,'QM01150','C11002','IS_A'),(69,'C10066','QM01110','IS_A'),(70,'TSK00101','TSK00001','IS_A'),(71,'HW01200','HW01000','IS_A'),(74,'C10070','QM01130','IS_A'),(75,'C10072','QM01130','IS_A'),(76,'C10071','QM01130','IS_A'),(77,'C10074','QM01150','IS_A'),(78,'HW01100','HW01000','IS_A'),(79,'C10073','QM01150','IS_A'),(80,'FF00010','C17501','IS_A'),(81,'C10076','QM01150','IS_A'),(82,'FF00011','C17501','IS_A'),(83,'C10075','QM01150','IS_A'),(84,'FF00012','C17501','IS_A'),(85,'C10077','QM01150','IS_A'),(87,'FF00014','C17504','IS_A'),(88,'FF00013','C17501','IS_A'),(89,'FF00016','C17501','IS_A'),(90,'FF00015','C17501','IS_A'),(92,'QM02000','PRM00401','IS_A'),(93,'C11002','QM01000','IS_A'),(94,'C11001','QM01000','IS_A'),(95,'C11004','QM01000','IS_A'),(96,'C11003','QM01000','IS_A'),(97,'C10047','QM01110','IS_A'),(98,'C10046','QM01110','IS_A'),(99,'C10045','QM01110','IS_A'),(100,'C10044','QM01110','IS_A'),(101,'C10043','QM01110','IS_A'),(102,'C10042','QM01110','IS_A'),(103,'C10041','QM01110','IS_A'),(104,'C10040','QM01110','IS_A'),(105,'C10048','QM01110','IS_A'),(106,'C10049','QM01110','IS_A'),(107,'QM01000','PRM00401','IS_A'),(108,'C10050','QM01110','IS_A'),(109,'C10056','QM01110','IS_A'),(110,'C10055','QM01110','IS_A'),(111,'C10058','QM01110','IS_A'),(112,'C10057','QM01110','IS_A'),(113,'C10052','QM01110','IS_A'),(114,'C10051','QM01110','IS_A'),(115,'C10054','QM01110','IS_A'),(116,'C10053','QM01110','IS_A'),(117,'C10059','QM01110','IS_A'),(119,'MTH12200','MTH12000','IS_A'),(120,'C12021','C12101','IS_A'),(121,'QM04004','QM04000','IS_A'),(122,'C12020','C12101','IS_A'),(123,'C12023','C12101','IS_A'),(124,'QM04002','QM04000','IS_A'),(125,'C12022','C12101','IS_A'),(126,'QM04003','QM04000','IS_A'),(127,'C12025','C12101','IS_A'),(128,'C12024','C12101','IS_A'),(129,'C12027','C12101','IS_A'),(130,'C12026','C12101','IS_A'),(131,'C12028','C12101','IS_A'),(132,'C12029','C12101','IS_A'),(133,'QM04001','QM04000','IS_A'),(134,'QM04000','PRM00401','IS_A'),(135,'MTH12000','MTH10000','IS_A'),(136,'C12030','C12101','IS_A'),(138,'C12034','C12101','IS_A'),(139,'C12033','C12101','IS_A'),(140,'C12032','C12101','IS_A'),(141,'C12031','C12101','IS_A'),(142,'C12038','C12101','IS_A'),(143,'C12037','C12101','IS_A'),(144,'C12036','C12101','IS_A'),(145,'C12035','C12101','IS_A'),(146,'C12039','C12101','IS_A'),(148,'SYS00101','PRM00621','IS_A'),(149,'SYS00102','PRM00621','IS_A'),(150,'SYS00103','PRM00621','IS_A'),(151,'C12041','C12101','IS_A'),(152,'C12040','C12101','IS_A'),(153,'C12047','C12101','IS_A'),(154,'C12046','C12101','IS_A'),(155,'C12049','C12103','IS_A'),(156,'MD00307','MD00300','IS_A'),(157,'MD00306','MD00300','IS_A'),(158,'C12043','C12101','IS_A'),(159,'C12042','C12101','IS_A'),(160,'C12045','C12101','IS_A'),(161,'C12044','C12101','IS_A'),(162,'C14002','MD00800','IS_A'),(163,'C14003','MD00800','IS_A'),(164,'C14001','MD00800','IS_A'),(165,'C14004','MD00800','IS_A'),(166,'SW00400','SW01200','IS_A'),(167,'C12052','C12101','IS_A'),(169,'C12051','C12101','IS_A'),(170,'C12050','C12101','IS_A'),(171,'C12056','C12103','IS_A'),(172,'C12055','C12103','IS_A'),(173,'C12054','C12104','IS_A'),(174,'C12053','C12101','IS_A'),(175,'QM05200','QM05000','IS_A'),(176,'MTH12100','MTH12000','IS_A'),(177,'PRM00031','PRM00001','IS_A'),(178,'QM05100','QM05000','IS_A'),(179,'QM05000','PRM00401','IS_A'),(184,'PRM00021','PRM00001','IS_A'),(186,'MD00400','PRM00301','IS_A'),(188,'MD00303','MD00300','IS_A'),(190,'MD00302','MD00300','IS_A'),(192,'MD00305','MD00300','IS_A'),(194,'QM05300','QM05000','IS_A'),(195,'MD00304','MD00300','IS_A'),(196,'MD00301','MD00300','IS_A'),(198,'MD00300','PRM00301','IS_A'),(199,'C40363','QM05100','IS_A'),(200,'C12002','C12101','IS_A'),(202,'C40362','QM05100','IS_A'),(203,'C12003','C12103','IS_A'),(204,'SW00300','SW01200','IS_A'),(206,'C40361','QM05100','IS_A'),(207,'C12004','C12101','IS_A'),(209,'C40360','QM05100','IS_A'),(210,'C12005','C12104','IS_A'),(211,'C12001','C12101','IS_A'),(212,'C12101','MTH00100','IS_A'),(213,'C12102','MTH00100','IS_A'),(214,'C12103','MTH00100','IS_A'),(215,'C12104','MTH00100','IS_A'),(216,'C16001','MTH11300','IS_A'),(219,'C16002','MTH11300','IS_A'),(220,'SW00500','SW01200','IS_A'),(222,'C16003','MTH11300','IS_A'),(223,'C40368','QM05100','IS_A'),(224,'C16004','MTH11300','IS_A'),(226,'C40369','QM05200','IS_A'),(227,'C16005','MTH11300','IS_A'),(229,'C40366','QM05200','IS_A'),(230,'C16006','MTH11300','IS_A'),(231,'C12007','C12101','IS_A'),(233,'C40367','QM05100','IS_A'),(234,'C16007','MTH11300','IS_A'),(235,'C12006','C12101','IS_A'),(237,'C40364','QM05100','IS_A'),(238,'C16008','MTH11300','IS_A'),(239,'C12009','C12101','IS_A'),(241,'C40365','QM05200','IS_A'),(242,'C16009','MTH11300','IS_A'),(243,'C12008','C12101','IS_A'),(244,'C40372','QM05100','IS_A'),(245,'C12015','C12101','IS_A'),(246,'C40371','QM05100','IS_A'),(247,'C12016','C12101','IS_A'),(248,'C40374','QM05300','IS_A'),(249,'C12013','C12103','IS_A'),(250,'C40373','QM05100','IS_A'),(251,'C12014','C12103','IS_A'),(253,'C12011','C12103','IS_A'),(254,'C12012','C12103','IS_A'),(255,'C40370','QM05100','IS_A'),(256,'C12010','C12103','IS_A'),(257,'C16011','MTH11300','IS_A'),(258,'C16012','MTH11300','IS_A'),(259,'C16010','MTH11300','IS_A'),(261,'C16015','MTH11300','IS_A'),(263,'C16016','MTH11300','IS_A'),(265,'C16013','MTH11300','IS_A'),(267,'C16014','MTH11300','IS_A'),(269,'C16019','MTH11300','IS_A'),(271,'C12019','C12101','IS_A'),(272,'C12018','C12101','IS_A'),(273,'C16017','MTH11300','IS_A'),(274,'C12017','C12101','IS_A'),(275,'C16018','MTH11300','IS_A'),(276,'C40339','QM05100','IS_A'),(277,'SW00119','SW01200','IS_A'),(278,'C40334','QM05100','IS_A'),(279,'OS00005','OS00001','IS_A'),(280,'PRM00650','PRM00648','IS_A'),(281,'SW00118','SW01200','IS_A'),(282,'C40333','QM05100','IS_A'),(283,'OS00004','OS00001','IS_A'),(285,'SW00117','SW01200','IS_A'),(286,'C40332','QM05100','IS_A'),(287,'OS00003','OS00001','IS_A'),(288,'SW00116','SW01200','IS_A'),(289,'C40331','QM05100','IS_A'),(290,'OS00002','SW01100','IS_A'),(291,'C40338','QM05100','IS_A'),(292,'OS00001','SW01100','IS_A'),(293,'C40337','QM05100','IS_A'),(294,'C40336','QM05100','IS_A'),(295,'C40335','QM05100','IS_A'),(297,'SW00110','SW01200','IS_A'),(298,'SW00111','SW01200','IS_A'),(299,'SW00114','SW01200','IS_A'),(300,'OS00008','OS00007','IS_A'),(301,'SW00115','SW01200','IS_A'),(302,'C40330','QM05100','IS_A'),(303,'OS00009','OS00007','IS_A'),(304,'SW00112','SW01200','IS_A'),(305,'OS00006','OS00001','IS_A'),(306,'OS00007','OS00001','IS_A'),(307,'C40329','QM05100','IS_A'),(308,'PRM00649','PRM00648','IS_A'),(309,'C40328','QM05100','IS_A'),(310,'PRM00647','PRM00633','IS_A'),(311,'PRM00648','PRM00621','IS_A'),(312,'PRM00645','PRM00634','IS_A'),(313,'PRM00646','PRM00634','IS_A'),(314,'PRM00644','PRM00634','IS_A'),(315,'C40321','QM05200','IS_A'),(316,'OS00014','OS00004','IS_A'),(317,'C40320','QM05100','IS_A'),(318,'OS00013','OS00004','IS_A'),(319,'C40323','QM05200','IS_A'),(320,'OS00016','OS00005','IS_A'),(321,'PRM00501','PRM00201','IS_A'),(322,'C40322','QM05200','IS_A'),(323,'OS00015','OS00005','IS_A'),(324,'C40325','QM05300','IS_A'),(325,'OS00010','OS00007','IS_A'),(326,'C40324','QM05200','IS_A'),(327,'C40327','QM05100','IS_A'),(328,'OS00012','OS00004','IS_A'),(329,'C40326','QM05300','IS_A'),(330,'OS00011','OS00007','IS_A'),(331,'SW00100','SW01200','IS_A'),(332,'OS00017','OS00005','IS_A'),(333,'OS00018','OS00005','IS_A'),(334,'OS00019','OS00005','IS_A'),(336,'SW00600','SW01200','IS_A'),(337,'SW00200','SW01200','IS_A'),(338,'PRM00632','SYS00101','IS_A'),(339,'PRM00633','SYS00101','IS_A'),(340,'PRM00634','SYS00101','IS_A'),(341,'PRM00635','SYS00101','IS_A'),(342,'SYS00300','PRM00601','IS_A'),(343,'SYS00201','SYS00200','IS_A'),(344,'OS00023','OS00002','IS_A'),(345,'SYS00202','SYS00200','IS_A'),(346,'C40359','QM05100','IS_A'),(347,'OS00022','OS00002','IS_A'),(348,'C40358','QM05100','IS_A'),(349,'OS00021','OS00002','IS_A'),(350,'C40357','QM05100','IS_A'),(351,'OS00020','OS00005','IS_A'),(352,'C40356','QM05100','IS_A'),(353,'C40355','QM05100','IS_A'),(354,'MD00500','PRM00301','IS_A'),(355,'C40354','QM05100','IS_A'),(356,'SYS00200','PRM00600','IS_A'),(357,'C40353','QM05100','IS_A'),(358,'OS00024','OS00002','IS_A'),(359,'PRM00631','PRM00621','IS_A'),(360,'C40351','QM05100','IS_A'),(361,'C40352','QM05100','IS_A'),(362,'C40350','QM05100','IS_A'),(363,'MD00700','PRM00301','IS_A'),(365,'PRM00621','PRM00601','IS_A'),(366,'C16022','MTH11300','IS_A'),(367,'C16021','MTH11300','IS_A'),(368,'C16020','MTH11300','IS_A'),(369,'C40347','QM05100','IS_A'),(370,'C40346','QM05100','IS_A'),(371,'C40349','QM05100','IS_A'),(372,'C40348','QM05100','IS_A'),(373,'C40343','QM05100','IS_A'),(374,'C40342','QM05100','IS_A'),(375,'C40345','QM05100','IS_A'),(376,'C40344','QM05100','IS_A'),(377,'C40340','QM05200','IS_A'),(378,'C40341','QM05100','IS_A'),(379,'C30019','FS01110','IS_A'),(380,'MTH11300','MTH11100','IS_A'),(381,'C30017','FS01110','IS_A'),(382,'C30018','FS01110','IS_A'),(383,'C30015','FS01110','IS_A'),(385,'C30016','FS01110','IS_A'),(386,'C30013','FS01110','IS_A'),(387,'C30014','FS01110','IS_A'),(388,'C30012','FS01110','IS_A'),(389,'C30011','FS01110','IS_A'),(390,'C17504','MD00900','IS_A'),(391,'C30010','FS01110','IS_A'),(392,'C17503','MD00900','IS_A'),(393,'C17502','MD00900','IS_A'),(394,'C17501','MD00900','IS_A'),(395,'MTH11200','MTH11000','IS_A'),(396,'MD00604','MD00600','IS_A'),(397,'C30006','FS01110','IS_A'),(398,'MD00603','MD00600','IS_A'),(399,'C30007','FS01110','IS_A'),(400,'C30008','FS01110','IS_A'),(401,'C30002','FS01110','IS_A'),(402,'C30003','FS01110','IS_A'),(403,'PRM00601','PRM00600','IS_A'),(404,'C30004','FS01110','IS_A'),(405,'PRM00600','PRM00101','IS_A'),(406,'C30005','FS01110','IS_A'),(407,'C30001','FS01110','IS_A'),(409,'C40315','QM05100','IS_A'),(410,'C40316','QM05100','IS_A'),(411,'C40313','QM05100','IS_A'),(412,'C40314','QM05100','IS_A'),(413,'C40311','QM05100','IS_A'),(414,'C40312','QM05100','IS_A'),(415,'MD00100','PRM00301','IS_A'),(416,'C40310','QM05200','IS_A'),(417,'C30037','FS01110','IS_A'),(418,'C30038','FS01110','IS_A'),(419,'MD00200','PRM00301','IS_A'),(420,'C30035','FS01110','IS_A'),(421,'C30036','FS01110','IS_A'),(422,'C40319','QM05100','IS_A'),(423,'C40317','QM05100','IS_A'),(424,'C30039','FS01110','IS_A'),(425,'C40318','QM05100','IS_A'),(426,'MD00601','MD00600','IS_A'),(427,'C30030','FS01110','IS_A'),(428,'MD00602','MD00600','IS_A'),(429,'MD00600','PRM00301','IS_A'),(430,'C30034','FS01110','IS_A'),(431,'C30033','FS01110','IS_A'),(432,'C30032','FS01110','IS_A'),(433,'C30031','FS01110','IS_A'),(434,'SW00130','SW01200','IS_A'),(435,'C40302','QM05100','IS_A'),(436,'C40303','QM05100','IS_A'),(437,'C40304','QM05100','IS_A'),(438,'C40305','QM05100','IS_A'),(439,'SW00127','SW01200','IS_A'),(440,'SW00128','SW01200','IS_A'),(441,'C40300','QM05100','IS_A'),(442,'C40301','QM05100','IS_A'),(443,'C30024','FS01110','IS_A'),(444,'C30025','FS01110','IS_A'),(445,'C30026','FS01110','IS_A'),(446,'C30027','FS01110','IS_A'),(447,'C40306','QM05100','IS_A'),(448,'C30028','FS01110','IS_A'),(449,'C40307','QM05100','IS_A'),(450,'C30029','FS01110','IS_A'),(451,'C40308','QM05100','IS_A'),(452,'C40309','QM05100','IS_A'),(453,'C30021','FS01110','IS_A'),(454,'C30020','FS01110','IS_A'),(455,'C30023','FS01110','IS_A'),(456,'C30022','FS01110','IS_A'),(457,'SW00124','SW01200','IS_A'),(458,'SW00123','SW01200','IS_A'),(460,'SW00126','SW01200','IS_A'),(461,'SW00125','SW01200','IS_A'),(462,'SW00120','SW01200','IS_A'),(463,'SW00122','SW01200','IS_A'),(464,'SW00121','SW01200','IS_A'),(742,'TSK10004','TSK10000','IS_A'),(468,'C30053','FS01110','IS_A'),(469,'C30054','FS01110','IS_A'),(470,'C13001','MD00200','IS_A'),(471,'C30055','FS01110','IS_A'),(472,'C13002','MD00200','IS_A'),(473,'C30056','FS01110','IS_A'),(474,'C13003','MD00200','IS_A'),(475,'C13004','MD00200','IS_A'),(476,'C10039','QM01110','IS_A'),(477,'C30050','FS01110','IS_A'),(478,'C13005','MD00200','IS_A'),(479,'C10038','QM01110','IS_A'),(480,'C30051','FS01110','IS_A'),(481,'C13006','MD00200','IS_A'),(482,'C10037','QM01110','IS_A'),(483,'C30052','FS01110','IS_A'),(484,'C13008','MD00200','IS_A'),(485,'C10035','QM01110','IS_A'),(486,'C13007','MD00200','IS_A'),(487,'C10036','QM01110','IS_A'),(489,'SYS04120','SYS04100','IS_A'),(490,'C10033','QM01110','IS_A'),(491,'C10034','QM01110','IS_A'),(492,'C10031','QM01110','IS_A'),(493,'C30058','FS01110','IS_A'),(494,'C10032','QM01110','IS_A'),(495,'C30057','FS01110','IS_A'),(496,'C10030','QM01110','IS_A'),(497,'C30059','FS01110','IS_A'),(498,'QM01100','QM01000','IS_A'),(499,'TSK00201','TSK00001','IS_A'),(500,'C30044','FS01110','IS_A'),(501,'MTH13000','MTH10000','IS_A'),(502,'C30045','FS01110','IS_A'),(503,'C30042','FS01110','IS_A'),(504,'C30043','FS01110','IS_A'),(505,'MTH13004','MTH13020','IS_A'),(506,'C10027','QM01110','IS_A'),(507,'C30040','FS01110','IS_A'),(508,'MTH13003','MTH13020','IS_A'),(509,'C10026','QM01110','IS_A'),(510,'C30041','FS01110','IS_A'),(511,'MTH13002','MTH13020','IS_A'),(512,'C10029','QM01110','IS_A'),(513,'MTH13001','MTH13000','IS_A'),(514,'C10028','QM01110','IS_A'),(515,'MTH13007','MTH13000','IS_A'),(516,'C10022','QM01110','IS_A'),(517,'SW00800','SW01200','IS_A'),(518,'C10023','QM01110','IS_A'),(519,'MTH13005','MTH13020','IS_A'),(520,'C10024','QM01110','IS_A'),(521,'MTH13006','MTH13000','IS_A'),(522,'C10025','QM01110','IS_A'),(523,'C30049','FS01110','IS_A'),(524,'C30048','FS01110','IS_A'),(525,'C10020','QM01110','IS_A'),(526,'C30047','FS01110','IS_A'),(527,'C10021','QM01110','IS_A'),(528,'C30046','FS01110','IS_A'),(529,'MTH11220','MTH11200','IS_A'),(530,'C10018','QM01140','IS_A'),(531,'C10017','QM01140','IS_A'),(532,'C10016','QM01140','IS_A'),(533,'C10015','QM01140','IS_A'),(534,'MTH13010','MTH13000','IS_A'),(535,'C10019','QM01140','IS_A'),(537,'C10010','QM01140','IS_A'),(740,'TSK10002','TSK10000','IS_A'),(539,'C10013','QM01140','IS_A'),(540,'C10014','QM01140','IS_A'),(541,'C10011','QM01140','IS_A'),(542,'C10012','QM01140','IS_A'),(741,'TSK10003','TSK10000','IS_A'),(544,'C11011','MD00400','IS_A'),(545,'MTH11210','MTH11200','IS_A'),(546,'C11012','MD00400','IS_A'),(547,'C11013','MD00400','IS_A'),(548,'C17250','C17503','IS_A'),(549,'C11014','MD00400','IS_A'),(550,'MTH13026','MTH13022','IS_A'),(551,'C10005','QM01120','IS_A'),(552,'C30062','FS01110','IS_A'),(553,'C17350','FF00014','IS_A'),(554,'MTH13025','MTH13022','IS_A'),(555,'C10004','QM01120','IS_A'),(556,'C30063','FS01110','IS_A'),(557,'MTH13024','MTH13022','IS_A'),(558,'C10007','QM01120','IS_A'),(559,'C30060','FS01110','IS_A'),(560,'MTH13023','MTH13022','IS_A'),(561,'C10006','QM01120','IS_A'),(562,'C30061','FS01110','IS_A'),(563,'QM03000','PRM00401','IS_A'),(564,'C17354','C17504','IS_A'),(565,'MTH13022','MTH13021','IS_A'),(566,'C10009','QM01120','IS_A'),(567,'MTH13021','MTH13020','IS_A'),(568,'C10008','QM01120','IS_A'),(569,'C17353','FF00014','IS_A'),(570,'C30064','FS01110','IS_A'),(571,'C17352','FF00014','IS_A'),(572,'MTH13020','MTH13000','IS_A'),(573,'C30065','FS01110','IS_A'),(574,'C17351','FF00014','IS_A'),(575,'C17151','FF00011','IS_A'),(576,'C17152','FF00011','IS_A'),(739,'TSK10001','TSK10000','IS_A'),(579,'MTH13029','MTH13021','IS_A'),(580,'C10000','C11001','IS_A'),(581,'C17155','FF00011','IS_A'),(582,'SYS04110','SYS04100','IS_A'),(583,'C10001','C11001','IS_A'),(584,'C17156','FF00011','IS_A'),(585,'MTH13027','MTH13021','IS_A'),(586,'C10002','C11001','IS_A'),(587,'C17153','FF00011','IS_A'),(588,'MTH13028','MTH13021','IS_A'),(589,'C10003','C11001','IS_A'),(590,'C17154','FF00011','IS_A'),(591,'C17159','FF00011','IS_A'),(592,'C17054','FF00012','IS_A'),(593,'C17157','FF00011','IS_A'),(594,'C17158','FF00011','IS_A'),(595,'C17052','FF00012','IS_A'),(596,'C17053','FF00012','IS_A'),(597,'C17050','FF00012','IS_A'),(598,'C17051','FF00012','IS_A'),(599,'MTH00010','PRM00201','IS_A'),(600,'MTH13030','MTH13021','IS_A'),(601,'MTH00011','PRM00201','IS_A'),(602,'MTH13031','MTH13030','IS_A'),(603,'MTH13032','MTH13030','IS_A'),(604,'MTH13033','MTH13030','IS_A'),(605,'MTH13034','MTH13030','IS_A'),(606,'MTH13035','MTH13007','IS_A'),(607,'MTH13036','MTH13007','IS_A'),(608,'MTH11000','MTH10000','IS_A'),(609,'C17002','FF00010','IS_A'),(610,'PRM00401','PRM00201','IS_A'),(611,'C17001','FF00010','IS_A'),(612,'C17004','FF00010','IS_A'),(613,'C17003','FF00010','IS_A'),(614,'C17006','FF00010','IS_A'),(615,'C17005','FF00010','IS_A'),(616,'C17008','FF00010','IS_A'),(617,'C17007','FF00010','IS_A'),(618,'MD00900','PRM00301','IS_A'),(619,'MTH11100','MTH11000','IS_A'),(620,'C17009','FF00010','IS_A'),(621,'SW00700','SW01200','IS_A'),(622,'SW00900','SW01200','IS_A'),(623,'TSK00301','TSK00001','IS_A'),(625,'C40200','MTH12200','IS_A'),(626,'C40201','MTH12100','IS_A'),(627,'C40202','MTH12100','IS_A'),(628,'C40203','MTH12100','IS_A'),(629,'C40204','MTH12100','IS_A'),(631,'C40205','MTH12100','IS_A'),(632,'C40206','MTH12100','IS_A'),(633,'MTH11120','MTH11100','IS_A'),(634,'C40207','MTH12100','IS_A'),(635,'C40104','MTH13007','IS_A'),(636,'C40208','MTH12100','IS_A'),(637,'C40209','MTH12100','IS_A'),(638,'C40100','MTH13007','IS_A'),(639,'C40101','MTH13007','IS_A'),(640,'C40102','MTH13007','IS_A'),(641,'C40103','MTH13007','IS_A'),(642,'MTH00020','PRM00201','IS_A'),(643,'MD00800','PRM00301','IS_A'),(644,'C40212','MTH12100','IS_A'),(645,'C17013','FF00010','IS_A'),(646,'C40213','MTH12100','IS_A'),(647,'C17012','FF00010','IS_A'),(648,'C40210','MTH12100','IS_A'),(649,'C17011','FF00010','IS_A'),(650,'C40211','MTH12100','IS_A'),(651,'C17010','FF00010','IS_A'),(652,'C40214','MTH12100','IS_A'),(653,'C40215','MTH12100','IS_A'),(654,'MTH11110','MTH11100','IS_A'),(655,'C17015','C17502','IS_A'),(656,'C17014','FF00010','IS_A'),(657,'ANL00002','ANL00001','IS_A'),(658,'ANL00003','ANL00001','IS_A'),(659,'ANL00004','ANL00001','IS_A'),(660,'ANL00005','ANL00001','IS_A'),(661,'ANL00006','ANL00001','IS_A'),(662,'ANL00007','ANL00001','IS_A'),(663,'ANL00008','ANL00001','IS_A'),(664,'ANL00009','ANL00001','IS_A'),(665,'ANL00010','ANL00001','IS_A'),(666,'ANL00011','ANL00001','IS_A'),(667,'ANL00012','ANL00001','IS_A'),(668,'ANL00013','ANL00001','IS_A'),(669,'ANL00014','ANL00001','IS_A'),(670,'ANL00015','ANL00001','IS_A'),(671,'ANL00016','ANL00001','IS_A'),(672,'ANL00017','ANL00001','IS_A'),(673,'ANL00018','ANL00001','IS_A'),(674,'ANL00019','ANL00001','IS_A'),(675,'ANL00020','ANL00001','IS_A'),(676,'ANL00021','ANL00001','IS_A'),(677,'ANL00022','ANL00001','IS_A'),(678,'ANL00023','ANL00001','IS_A'),(679,'ANL00024','ANL00001','IS_A'),(680,'ANL00025','ANL00001','IS_A'),(681,'ANL00026','ANL00001','IS_A'),(682,'ANL00027','ANL00001','IS_A'),(683,'ANL00028','ANL00001','IS_A'),(735,'ANL00042','ANL00028','IS_A'),(734,'ANL00041','ANL00028','IS_A'),(733,'ANL00040','ANL00028','IS_A'),(732,'ANL00039','ANL00028','IS_A'),(731,'ANL00038','ANL00028','IS_A'),(730,'ANL00037','ANL00028','IS_A'),(729,'ANL00036','ANL00028','IS_A'),(728,'ANL00035','ANL00028','IS_A'),(727,'ANL00034','ANL00028','IS_A'),(726,'ANL00033','ANL00028','IS_A'),(725,'ANL00032','ANL00028','IS_A'),(724,'ANL00031','ANL00028','IS_A'),(723,'ANL00030','ANL00028','IS_A'),(722,'ANL00029','ANL00028','IS_A'),(698,'ANL00043','ANL00001','IS_A'),(699,'ANL00044','ANL00001','IS_A'),(700,'ANL00045','ANL00001','IS_A'),(701,'ANL00046','ANL00001','IS_A'),(702,'ANL00047','ANL00001','IS_A'),(703,'ANL00048','ANL00001','IS_A'),(704,'ANL00049','ANL00001','IS_A'),(705,'ANL00050','ANL00001','IS_A'),(706,'ANL00051','ANL00001','IS_A'),(707,'ANL00052','ANL00001','IS_A'),(708,'ANL00053','ANL00001','IS_A'),(709,'ANL00054','ANL00001','IS_A'),(710,'ANL00055','ANL00001','IS_A'),(711,'ANL00056','ANL00001','IS_A'),(712,'ANL00057','ANL00001','IS_A'),(713,'ANL00058','ANL00001','IS_A'),(714,'ANL00059','ANL00001','IS_A'),(721,'ANL00061','ANL00059','IS_A'),(720,'ANL00060','ANL00059','IS_A'),(717,'ANL00062','ANL00001','IS_A'),(718,'ANL00063','ANL00001','IS_A'),(719,'ANL00064','ANL00001','IS_A'),(736,'TSK01000','TSK00001','IS_A'),(737,'ANL00001','MTH10000','IS_A'),(738,'HW01300','HW01000','IS_A');
/*!40000 ALTER TABLE `RELATIONSHIP` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `SEMANTIC_TYPE`
--

DROP TABLE IF EXISTS `SEMANTIC_TYPE`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `SEMANTIC_TYPE` (
  `TID` int(11) NOT NULL,
  `STR` varchar(255) DEFAULT NULL,
  PRIMARY KEY (`TID`),
  UNIQUE KEY `UQ_SEMANTIC_TYPE_TID` (`TID`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `SEMANTIC_TYPE`
--

LOCK TABLES `SEMANTIC_TYPE` WRITE;
/*!40000 ALTER TABLE `SEMANTIC_TYPE` DISABLE KEYS */;
/*!40000 ALTER TABLE `SEMANTIC_TYPE` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2014-02-14 14:58:47
